---
title: "MICB425_portfolio"
author: "Yaqian Luo (59751503)"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output:
  html_document:
    toc: yes
---
# Module_01: Origins and Earth Systems  

## Module_01_Evidence_Worksheet_01: "Prokaryotes: The Unseen Majority"

#### Reading
W. B. Whitman, D. C. Coleman, and W. J. Wiebe, "Prokaryotes: The unseen majority," *PNAS*, vol. 95, pp. 6578-6583, June 1998. (http://www.pnas.org/content/95/12/6578)

#### Learning Objectives
Describe the numerical abundance of microbial life in relation to ecology and biogeochemistry of Earth systems.


#### General Questions
***What were the main questions being asked?***

The questions are: What are the actual number of prokaryotes? and what total amount of their carbon content on earth?


***What were the primary methodological approaches used?***

The primary methodological approaches used in the paper were classification, literature survey, and comparison. Classification was for the estimation of prokaryotic abundance, literature survey was for data collection, and comparison method was for comprehension of the estimated results. 

Due to ubiquity of prokaryote on Earth, classification of its habitats was helpful to estimate the number of prokaryotes and the amount of carbon content in microbes. Prokaryotes’ habitats generally include aquatic environments, soil, subsurface, and others like animals, leaves, and air. The first three habitats were concerned, because they had been proved in the paper to be the major place for prokaryotes. For each of them, they were divided into several groups for estimation. For example, subsurface consisted of marine subsurface and terrestrial subsurface, and different methods and values were applied to determine the number of prokaryotes and carbon content in different kind of subsurface. 

Literature survey was the major approach for data collection, because it was the simplest and quickest method to obtain data needed. For the calculations of number of prokaryotes, values of cellular density, habitat volume or area, and relative parameters were collected from abundant literatures. 

Comparison was another major approach used in the paper. The estimated results were huge and kind of beyond our imagination. Comparison was an effective way to help readers comprehend these enormous numbers by associating them with those numbers easy to understand. In the paper, prokaryote was contrasted with plant, in order to justify the abundance of prokaryotic carbon and nutrients.    

Cellular abundance was estimated with cellular density and volume/area in different environments based on specific assumptions. The data of cellular density and volume/area was collected from literature. Carbon content was calculated through determination of average cellular carbon and the number of prokaryotes in different habitats.  

***Summarize the main results or findings.***

The number of prokaryotes and total amount of their cellular carbon on earth are 4-6×10^30^ cells and 350-550 Pg of C, respectively. Prokaryotic carbon are substantial, about 60-100% of total carbon in plants. Aquatic environments, soil, and subsurface are three dominant habitats for earth’s prokaryotes, in which the number of prokaryotic cells are calculated to be 1.2×10^29^, 2.6×10^29^, 3.75-6.0×10^30^, respectively, the amount of carbon content in prokaryotes are 2.2 Pg of C, 26 Pg of C, and 325-519 Pg of C, respectively, and turnover time for heterotrophic prokaryotes is 6-25 days, 0.8 yr, and 2.5 yr, respectively. The annual cell production for prokaryotes is highest in the open ocean.


***Do new questions arise from the results?***

Some new questions are raised based on the results of the study. They are:

1. It has already been proved in the paper that prokaryotes are abundant and have huge carbon and nutrient contents. However, it is still not clear that whether prokaryote is the largest living reservoir for carbon and essential elements on earth.

2. Based on the results of turnover time for prokaryotes in different habitats, a question needs to be answered that why the turnover time for prokaryote in subsurface is far longer than that in other ecosystem.

3. Prokaryotes have huge potential to obtain genetic diversity and a large number of species, due to their large population size and quick turnover time. Nevertheless, we are not sure that whether the current definition of a prokaryotic species is consistent with the real definition, and there may be some taxonomic tribes being grouped into the same species. 


***Were there any specific challenges or advantages in understanding the paper (e.g. did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?*** 

There were several advantages and challenges in comprehending the paper:

1. Habitats were redefined in the paper, which was helpful for understanding methods used in the estimation and making more accurate calculations. For example, subsurface in the paper refered to terrestrial habitats below 8 m and marine sediments below 10 cm, not including upper 10 cm of marine sediments that was a part of aquatic environments according to the paper.

2. The estimation of prokaryotic abundance and turnover time involved a lot of calculation. However, formulas for calculations were not presented in the paper, making it difficult for readers to understand the results and researchers to replicate the study. For example, it was hard to figure out how to calculate the generation time for prokaryotes in upper 200m of the ocean based on the efficiency of carbon assimilation and the amount of net productivity without any relevant formulas in the paper.

3. Assumptions were often made during the estimation of prokaryotic abundance and carbon contents. They were useful to obtain final results, but also show deficiencies of calculation approaches need to be solved in the future studies. For instance, there were two methods introduced for the estimation of terrestrial subsurface prokaryotes’ abundance, both of which made some assumptions. The average porosity of terrestrial subsurface was assumed to be 3% in the first approach, while the ratio of unattached prokaryotes in aquifer was considered to be consistent for the second approach. As a result, it was reported that the number of prokaryotes in terrestrial subsurface fell into a range of 2.5-25×10^29^ cells, which needed more accuracy in the further research.  

## Module_01_Evidence_Worksheet_02: "Life and the Evolution of Earth’s Atmosphere"" 

#### Reading:  
J. F. Kasting and J. L. Siefert, "Life and the Evolution of Earth's Atmosphere," *Science*, vol. 296, pp. 1066-1068, May 2002. (http://science.sciencemag.org/content/296/5570/1066) 

#### Learning objectives:   
Comment on the emergence of microbial life and the evolution of Earth systems
    
    
* Indicate the key events in the evolution of Earth systems at each approximate moment in the time series. If times need to be adjusted or added to the timeline to fully account for the development of Earth systems, please do so. 

    + 4.6 billion years ago  
Earth was born. 

    + 4.5 billion years ago  
Moon was formed which influenced the spin and tilt of Earth;  
A rock vapor atmosphere was formed immediately after the birth of Moon. 

    + 4.4 billion years ago  
The oldest minerals known on Earth are zircons formed 4.4-Gyr;  
Ocean and early atmosphere were formed.
    
    + 4.1 billion years ago  
Life might occur.
    
    + 4.0 billion years ago  
The oldest rock, Acasta gneiss, was formed 4.0 billion years ago. 

    + 3.8 billion years ago  
Massive meteorite impacts halted;  
Life was sustained by anoxygenic photosynthesis;    
Methanogens might evolve. 

    + 3.5 billion years ago    
Life must be presented on Earth;  
Oxygenic photosynthesis and the evolution of cyanobateria occurred;  
Biochemical carbon cycle started to perform.

    + 3.2 billion years ago  
Filamentous appeared on Earth.
    
    + 3.0 billion years ago  
Microbial life was widespread in the mid-Archaean.

    + 2.7 billion years ago  
Biological lipids were shown in 2.7-Gyr rocks;  
ancestral eukaryotes appeared.
    
    + 2.2 billion years ago  
Rocks were redbeds;  
Complex eukaryotes might show up;    
The cellular cybernetic switched between mitochondria and chloroplasts;  
Earth became a snowball at first time.

    + 2.1 billion years ago   
Multicellular life appeared.

    + 1.3 billion years ago  
The ancestor of animals started to evolve. 
    
    + 550,000 years ago  
Humans appeared.

    + 200,000 years ago
The second most recent glacial period began. 

* Describe the dominant physical and chemical characteristics of Earth systems at the following waypoints:  

    + **Hadean**  
    1. Earth was rich in water vapor and carbon dioxide, and tempertature was above 500℃ when it was born.  
    2. Earth radiated heat to space.  
    3. Surface temperature lingered near 100℃ with a steam greenhouse until 4 billion years ago.  
    4. Earth was glacial with intervals of inferno after the meteorite impacts. 
    
    + **Archean**  
    1. The oceans was heated over 110℃, and seawater chemistry was always changing during meteorite impacts.
    2.In the early Archaean, there was a relatively oxidized atmosphere-ocean system and a reduced fuilds in contact with mantle-derived magmas.
    3. Greenhouse gases (methane and carbon dioxide) were predominant.
    4. The atmosphere was rich in sulphur gases and carbon dioxide with partial pressure over 2 bars.  
    5. Sulphate was presented in 3.5-Gyr rocks.  
    6. Minerals were contained in many Archaean rocks like detrital pyrite, siderite and uraninite.  
    
    + **Proterozoic**  
    1. The oxygen level increased sharply about 2.2 billion years ago.  
    2. The atmosphere became oxic.
    
    + **Phanerozoic**
    1. oxygenation of atmosphere increased.  
    2. The top of troposphere is cold and water falls back to Earth's surface.  
    3. Stratosphere is dry with little hydrogen at the top. 
    4. Greenhouse gases like water vapor and carbon dioxide warms up the surface of Earth. 
    
## Module_01_Evidence_Worksheet_03: “The Anthropocene”  


#### Learning objectives:  

Evaluate human impacts on the ecology and biogeochemistry of Earth systems.

#### Reading:  
C. N. Waters, J. Zalasiewicz, C. Summerhayes, A. D. Barnosky, C. Poirier, A. Gałuszka, A. Cearreta, M. Edgeworth, E. C. Ellis, M. Ellis, C. Jeandel, R. Leinfelder, J. R. McNeill, D. deB. Richter, W. Steffen, J. Syvitski, D. Vidas, M. Wagreich, M. Williams, Z. An, J. Grinevald, E. Odada, N. Oreskes, A. P. Wolfe, "The Anthropocene is functionally and stratigraphically distinct from the Holocene," Science, vol.351, issue 6269, pp. 137, Jan. 2016.

#### General Questions:  

***What were the main questions being asked?***

Following questions are being asked in the paper: 

1. Have humans changed the Earth to an extent that recent and currently forming geological deposits is different from those of the Holocene and early epochs?

2. When did these stratigraphic signals become recognizable worldwide?

3. How are stratigraphic units formally recognized earlier in the Quaternary period?

4. How are stratigraphic signals identified and used in stratigraphic characterization and correlation of Anthropocene epoch?

***What were the primary methodological approaches used?***

The major methodological approaches in this research were classification and literature survey. The stratigraphic signals of Anthropocene were generally classified into several categories based on locations where signals showed up and their influences: sediments and ice, climate change, and biosphere. Each stratigraphic signal was proved by compelling evidence in literature and comparison with historical records in Holocene or early epochs. For example, the concentration of atmospheric CO~2~ in air bubbles in Antarctic glacial ice for early geological time and current time were collected and described in the paper, which indicated a stratigraphic signature of constant rise in CO~2~ concentration since around 1850.  

***Summarize the main results or findings.***

Quaternary stratigraphic units are defined based on climate change events, similarly to the determination of three subdivisions in Holocene epoch. The driving human forces responsible for the anthropogenic signatures are a combination of the forces of accelerated technological development, rapid growth of human population, and increased consumption of resources. The stratigraphic signals correlated to Anthropocene epoch are the appearance of manufactured material in sediments, the modification of biogeochemical cycles, climate change, and biotic changes.

Stratigraphic signals of Anthropocene epoch have been shown in sediments and ice. New anthropogenic materials and deposits, such as aluminum, concrete, plastics, unburned particles produced during fossil fuel combustion, have had huge production since the 20th century. They will leave identifiable fossil and geochemical records in sediments or permanent markers in both sediments and glacial ice. On the other hand, human activities including land use for residence and industries, deforestation, and dam constructions, have significantly directly or indirectly modified sedimentary processes over the past several decades. Besides, the geochemical signatures introduced by human, like increased concentrations of pesticide residues, nutrients and industrial metals in soil, have stratigraphic consequences. Moreover, radiogenic and carbon signatures have been discovered in sediments and ice respectively.  

Currently abnormally global warming and high global sea levels, caused by increasing anthropogenic emission of greenhouse gases, are considered as other robust stratigraphic signals of Anthropocene. Additionally, the anthropogenic energy imbalance amplifies stratigraphic signals related to climate change and sea-level rise. 

Accelerated biotic change is another stratigraphic signal for Anthropocene. A notable increase can be observed in extinction rates of species from the 19th century, and species assemblages and relative abundance have been altered globally due to species invasion and human activities. 

***Do new questions arise from the results?***

There are several questions needed to further investigate based on the findings in the research:

1.	In which way, the Anthropocene can be defined? By GSSA , GSSP or both?

2.	When did the Anthropocene exactly start?

3.	Is it necessary to formalize the Anthropocene? Or would it be better to leave it as an informal but solidly founded geological time term like Precambrian?

***Were there any specific challenges or advantages in understanding the paper (e.g. did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?***

Abundant evidence from literature was provided in the paper to support each of stratigraphic signals. This information constructed strong links between these stratigraphic signatures and Anthropocene. For instance, the emergence of new anthropogenic materials in sediments reflected the difference between Anthropocene and earlier geological time like Holocene. Climate change pattern in historical records was describe in the paper to show the abnormality of current global warming, another possible signal of Anthropocene. Therefore, conclusion that we are experiencing a new geological time called as Anthropocene was strongly justified based on this compelling evidence.

A figure summarizing the time when these stratigrapic signals appear would be more helpful for the understanding of Anthropocene’s characteristics and determination of the start of the new geological time. Additionally, although trends of some signals over the past decades were obvious in many of figures shown in the paper, such as the increasing CO2 concentration from the ice core and high species extinction rate, it would be more convincing if statistical analysis was conducted to check significant difference of Anthropocene from previous geological time. 

## Module_01_Problem_Set_01: “Prokaryotes: The unseen majority”

#### Reading:  
Whitman WB, Coleman DC, Wiebe WJ. 1998. Prokaryotes: The unseen majority. *PNAS*, vol. 95:6578-6583

#### Learning objectives: 
Describe the numerical abundance of microbial life in relation to the ecology and biogeochemistry of Earth systems.

#### Specific Questions:

***What are the primary prokaryotic habitats on Earth and how do they vary with respect to their capacity to support life? Provide a breakdown of total cell abundance for each primary habitat from the tables provided in the text.*** 

*Qrestion 1*   
The primary prokaryotic habitats are aquatic environments, soil, and subsurface.

*Question 2*  
Aquatic environments: Prokaryotic cell abundance is in huge variance under different aquatic conditions. The average of cellular density for prokaryotes decreases from 5 ×10^5^ cells/ml in upper 200m of the open ocean to 5 ×10^4^ cells/ml where the depth is over 200m. Although cellular density in freshwaters and saline lakes is about 106 cells/ml and higher than that in seawater, the total cell number of prokaryotes in these two kinds of aquatic ecosystems is three orders of magnitude below the number in seawaters, due to small volume compared to seawater. Cell abundance in polar region contributes to a small portion of the total cellular number in aquatic environments.

Soil: Huge difference in prokaryotic cell abundance also exists among different types of soils. The number of prokaryotes in forest soils is much less than that in others among which there is little variance.

Subsurface: cell density declines with depth in unconsolidated subsurface sediments. It is estimated in the paper that 97% of prokaryotic cells in unconsolidated sediment occur at depth no more than 600m.  

*Question 3*  
The total number of prokaryotic cells on Earth is 415-640×10^28^. The number in aquatic habitats, soil and subsurface are 11.8×10^28^, 25.6×10^28^, and 380-605×10^28^, respectively.

***What is the estimated prokaryotic cell abundance in the upper 200 m of the ocean and what fraction of this biomass is represented by marine cyanobacterium including Prochlorococcus? What is the significance of this ratio with respect to carbon cycling in the ocean and the atmospheric composition of the Earth?***

The estimated prokaryotic cell abundance in the upper 200m of the ocean is 3.6×10^28^ cells.

The portion of marine cyanobacterium is 8% calculated as below:
4×10^4^ cells/ml ÷ 5×10^5^ cells/ml × 100% = 8%

This ratio affects the capability of ocean to capture carbon dioxide from atmosphere, since cyanobaterium obtains energy by photosynthesis which is a process involving carbon dioxide as a reactant. Cyanobacterium also has huge contributions to the formation of modern atmosphere. They are a net source of oxygen, and even in some cases they are the only source of oxygen. This kind of prokaryotes is also responsible for nitrogen fixation [1]. Therefore, cyanobacterium plays a significant role in the atmospheric composition. 


***What is the difference between an autotroph, heterotroph, and a lithotroph based on information provided in the text?***

The way to obtain materials to support its own life is different between autotroph, heterotroph, and lithotroph. Autotroph is able to produce complex organic compounds using inorganic carbon (like carbonate) and energy from light or chemical reactions. For heterotroph, organic carbon is their carbon as well as energy source. Lithotroph obtains energy from inorganic substrate (such as ammonium).


***Based on information provided in the text and your knowledge of geography what is the deepest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this depth?***

The deepest habitat may be the subsurface with a depth of 15 km below sea level and 4 km below the Mariana Trench (about 11 km below sea level [2]). 

Temperature is the primary limiting factor under this circumstance, because temperature at this depth is about 125℃, which is close to maximal temperature for prokaryotic life.


***Based on information provided in the text your knowledge of geography what is the highest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this height?***

The highest habitat for prokaryotes may be in the atmosphere at an altitude of 57-77 km. 

The primary limiting factor in this case may be nutrient and UV. There are several essential materials for prokaryotic life, such as phosphate, which are scarce at this altitude. UV is another inhibitive factor to prokaryotic life because it can destroy DNA or RNA chain and it is not absorbed at this altitude due to lack of ozone in atmosphere [3]. Therefore, nutrient and UV may be the dominant factors for prokaryotic life at the highest habitat.

***Based on estimates of prokaryotic habitat limitation, what is the vertical distance of the Earth’s biosphere measured in km?***

The deepest habitat for prokaryotic life may be around 15 km below the sea level and the highest habitat may be 57-77 km above the ground. So the vertical distance of biosphere on Earth is about 72-92 km.

***How was annual cellular production of prokaryotes described in Table 7 column four determined? (Provide an example of the calculation)***

The cellular production rate is calculated using the formula below:  
cellular production rate = Population size × 365 days/yr ÷ turnover time

e.g.  For Marine heterotrophs in the upper 200 m of the open sea, there is:  
cellular production rate = 3.6×10^28^ cells × 365 days/yr ÷ 16 days = 8.2×10^29^ cells/yr


***What is the relationship between carbon content, carbon assimilation efficiency and turnover rates in the upper 200m of the ocean? Why does this vary with depth in the ocean and between terrestrial and marine habitats?***

*Question 1*  
Large amount of carbon available for prokaryotes and high carbon assimilation efficiency accelerate prokaryotes’ turnover rates. However, prokaryotes in the ocean need more carbon and their turnover rates decrease when carbon assimilation efficiency is low. 

The turnover rate of prokaryotes living in the upper 200m of the ocean is calculated as follows:  
Total amount of cellular carbon for prokaryotes in the upper 200m of the ocean is :  
3.6×10^28^ cells × 20 fg C/cell = 0.72 Pg C  
The amount of net productivity to support prokaryotes is:  
0.72 Pg C × 4 = 2.88 Pg C  
Net primary carbon productivity in marine consumed by prokaryotes in the upper 200m is:  
51 Pg C/yr × 85% = 43 Pg C/yr  
Turnover rate for heterotrophs in upper 200m ocean is:  
43 Pg C/yr ÷ 2.88 Pg C = 14.9/yr  

*Question 2*  
The average turnover time rises with depth in the ocean. One reason may be that temperature in the deep ocean is lower than that in the upper ocean, which declines the efficiency of carbon assimilation. Another possible cause is the amount of carbon content needed to support prokaryotes’ life increases with depth in the ocean, since cell abundance in deep oceanic water is larger than that in the upper ocean. As a result, the average turnover time for prokaryotes rises in deeper ocean. 

The calculated average turnover time for prokaryotes living in terrestrial habitats is far larger than that in marine environment. There are several possible explanations for the difference. Firstly, the required amount of carbon content for prokaryotes in terrestrial habitats may be overestimated, because there are other metabolic activities detected like sulfate reduction and lithoautotrophic processes for the growth of prokaryotes in terrestrial habitat. Secondly, subsurface biomass may be overestimated and carbon assimilation efficiency may be underestimated when turnover rates of prokaryotes in subsurface is calculated. Therefore, the calculated result for terrestrial habitats may be larger than the real value.   

***How were the frequency numbers for four simultaneous mutations in shared genes determined for marine heterotrophs and marine autotrophs given an average mutation rate of 4 x 10^-7^ per DNA replication? (Provide an example of the calculation with units. Hint: cell and generation cancel out)***

The calculation for average mutation rate is presented below.  
Mutation frequency (/yr) = mutation rate × cell abundance× turnover rate (/yr)  
Mutation time (hr) = 365 day/yr × 24 hr/day ÷ mutation frequency (/yr)  

Take marine heterotrophs in the upper 200 m as an example:   
Mutation rate for four simultaneous mutation is:  
(4×10^-7^)^4^ = 256×10^-28^ per generation  
Mutation frequency is:  
256×10^-28^per generation × 3.6 ×10^28^ cells × 365 day/yr ÷ 16 day = 21024/yr  
Mutation time is:  
365 day/yr × 24 hr/day ÷ 21024/yr = 0.4 hr


***Given the large population size and high mutation rate of prokaryotic cells, what are the implications with respect to genetic diversity and adaptive potential? Are point mutations the only way in which microbial genomes diversify and adapt?***

High mutation rate leads to large genetic diversity and high potential to form new species. Large population size and high mutation rate increase prokaryotes’ ability to survive in extreme environments [4].

Point mutations are not the only way for microorganisms to obtain genomic diversity and new phenotypes. For eukaryotic microorganisms like fungi and green algae, DNA in their cells is arranged in complex chromosomes. Hence large-scale mutations like amplifications and inversions are another approach for eukaryotic microorganisms to increase their genomic diversity. However, point mutations are the only way for prokaryotes to diversify their genomes and gain new adaptation, because they do not have chromosomal structure.


***What relationships can be inferred between prokaryotic abundance, diversity, and metabolic potential based on the information provided in the text?***

There is a large diversity of prokaryotes on Earth due to their large abundance and high mutation rate. Large abundance and high mutation rate increase the probability for prokaryotes to form new genetic sequence, as a result, may obtain new phenotypes [4]. After several genetic mutations, a new species is generated, which can be completed within a short period of time like days or months. Prokaryotes present on Earth for millions of years. Thus, they have evolved into millions of species, some of which have not been discovered yet.  

Due to the high prokaryotic diversity, their metabolic activities are different from each species. Autotrophs, heterotrophs, and lithoautotrophs are three major metabolisms for prokaryotes, which enable them to use a wide range of materials to support their life, such as methanol, iron and ammonium, and survive in anaerobic and aerobic environments and a variety of ecosystems including marine, soil and subsurface [4].   

Although single cell is very small and its metabolic potential is negligible, the metabolic potential of all kinds of prokaryotic cells cannot be ignored due to the huge number of prokaryotic cells. They have huge impacts on Earth’s biogeochemical transformation. For example, prokaryote in the ocean is one of the major atmospheric oxygen sources and has huge effects on the nitrogen cycle. 

#### References  
[1] James F. Kasting and Janet L. Siefert. 2002. Life and the Evolution of Earth’s Atmosphere. Environmental Microbiology, 296: 1066-1068.  
[2] Deepest Part of the OceanI (n.d.)[Online]. Available: https://geology.com/records/deepest-part-of-the-ocean.shtml.  
[3]Ozone layer (n.d.) [Online]. Available: https://en.wikipedia.org/wiki/Ozone_layer.  
[4] Whitman WB, Coleman DC, Wiebe WJ. 1998. Prokaryotes: The unseen majority. PNAS, vol. 95:6578-6583.

## Module_01_Problem_set_02: “Microbial Engines”

#### Reading:  
P. G. Falkowski, T. Fenchel, E. F. Delong, "The Microbial Engines That Drive Earth’s Biogeochemical Cycles," *Science*, vol.320, pp. 1034-1039, May 2008. (http://science.sciencemag.org/content/320/5879/1034.full)

#### Learning objectives: 
Discuss the role of microbial diversity and formation of coupled metabolism in driving global biogeochemical cycles.

#### Specific Questions:

***What are the primary geophysical and biogeochemical processes that create and sustain conditions for life on Earth? How do abiotic versus biotic processes vary with respect to matter and energy transformation and how are they interconnected?***

*Question 1*

Tectonics and atmospheric photochemical processes are two primary geophysical processes on Earth. The biological fluxes of the six major elements-H, C, N, O, S, and P-are the major biogeochemical processes.

*Question 2*

The chemical reactions abiotic and biotic processes depend on are different. Abiotic processes are based on acid-base chemistry involving formation and break of chemical bonds and transfers of protons without electrons. Biotic processes are driven by redox reactions, in which electrons and protons are both transferred. Not all of biotic processes are directly dependent on chemical bond energy. One example is photosynthesis. In terms of mass transformation, the importance of these two types of processes on biological chemical cycling is not the same. Biotic processes play significant roles in the biological fluxes of H, C, N, O, and S, while abiotic processes have contributions to the fluxes of C, S, and P.

*Question 3*

Both abiotic and biotic processes are responsible for the biogeochemical cycle of elements. Abiotic processes like erosion, geothermal activity and abiotic oxidation are able to bring the elements in sediments and atmosphere back to the biogeochemical cycle which is largely driven by biotic processes. For example, through geothermal processes, sulfate as HS^-^ and SO~3~^2-^ is regenerated into biochemical processes from Earth’s mantle and crust, and then contribute to the biotic processes of SO~4~^2-^/S respiration and anoxygenic photosynthesis.


***Why is Earth’s redox state considered an emergent property?***

Abiotic and biotic processes are formed during the evolution of biogeochemical cycle. They work together and change the redox condition of Earth's surface. As a result, the average redox state of Earth is created by feedbacks between geophysical and biological processes. 
  

***How do reversible electron transfer reactions give rise to element and nutrient cycles at different ecological scales? What strategies do microbes use to overcome thermodynamic barriers to reversible electron flow?***

*Question 1*

Element and nutrient cycles are maintained by thermodynamically favorable pathways and interactions among a variety of microbial species. 

Reversible reactions apply thermodynamically favorable pathways to sustain elements cycle under different conditions. For example, in some cases where hydrogen is sufficient, methane is generated by methanogenic Archaea through the reversible reaction using CO~2~ and H~2~ as reactants. However, at the places lack of hydrogen, methanogens consume methane because the reverse reaction is favorable in this case. 

Additionally, reversible reactions are sometimes catalyzed by the interactions between microorganisms from different species which may be temporally and spatially separated. For instance, the biogeochemical cycle of nitrogen is accomplished via cooperation of diverse microbes, including aerobic ammonium oxidizing bacteria and anaerobic denitrification bacteria. 

*Question 2*

Microorganisms use identical or near-identical pathways and cooperate with other species to overcome thermodynamic barriers. Identical or near-identical pathways, which are thermodynamically favorable in reversible reactions, may be employed. These pathways are easier for microbes to overcome thermodynamic barriers compared with their reverse processes. For instance, when hydrogen is insufficient, methane is oxidized by microbes to hydrogen, which is reverse to the formation reaction of methane performed by methanogenic Archaea. 

Besides, microbes sometimes may cooperate with multispecies for biogeochemical transformation, so that they are able to overcome thermodynamic barriers with the help from other microorganisms. Green sulfur bacteria and some Archaebacteria work with organisms able to perform citric acid cycle to get energy they need for carbon dioxide assimilation. 

***Using information provided in the text, describe how the nitrogen cycle partitions between different redox “niches” and microbial groups. Is there a relationship between the nitrogen cycle and climate change?***

*Question 1*

The biogeochemical nitrogen cycle is achieved by three sets of microbes. The first set of microbial group reduces N~2~ in atmosphere to NH4+ anaerobically, catalyzed by an enzyme called nitrogenase. The second set of microorganisms is responsible for the oxidization of NH~4~^+^ to NO~3~^-^ in a two-stage pathway under presence of oxygen. NH~4~^+^ is oxidized to NO~2~^-^ by ammonium oxidizing bacteria, and then NO~2~^-^ is subsequently oxidized to NO~3~^-^ by nitrate oxidizing bacteria. The third set of microbes is able to utilize NO~3~^-^ and NO~2~^-^ as electron acceptor and transform organic carbon into CO~2~ without oxygen. This process produces N~2~ to atmosphere. Therefore the nitrogen cycle is closed.  

*Question 2*

There are interactions between nitrogen cycle and climate change. Nitrogen cycle has direct and indirect impacts on global warming. N~2~O, a powerful greenhouse gas, is generated during the processes of nitrification as a byproduct and denitraification as an intermediate product [2]. The emission of N~2~O to atmosphere accelerates the rate of global warming. Besides, NO~x~, which is also emitted in nitrogen cycle, indirectly affect radiative forcing through effects on atmospheric concentrations of CH~4~, O~3~ and aerosol particulate matter [3]. However, the emission leads to a net cooling effect on a short time. 

Climate change is able to change nitrogen fluxes in the globally biogeochemical cycle, since it is able to alter biochemical processes involved in nitrogen cycle as well as the amount of nitrogen released to water or air. Extreme precipitation events flush more fertilizer or manure into freshwater and reduce the time for denitrification because runoff is speeded up [3]. Consequently, the amount of nitrogen in aquatic environments increases and the release of nitrogen gas to atmosphere declines. 


***What is the relationship between microbial diversity and metabolic diversity and how does this relate to the discovery of new protein families from microbial community genomes?***

*Question 1*

Microbial diversity may not lead to metabolic diversity. According to the modes of evolution described in the text, diverse microbial groups may perform the same metabolic pathway due to horizontal gene transfer. For example, nitrogenases genes have been widely distributed among different microbial groups on Earth, as well as methanogenesis genes. Both of them are essential to biogeochemical element cycles.    

However, by evaluating the relative depth of protein “sequence space”, the number of protein families in microbial groups is found to be linear proportional to genome size which is correlated to evolutionary rate. 

*Question 2*

Because microbial diversity is dependent on genome diversity and the number of novel protein families is linearly proportional to genome size, the number of newly discovered protein families is also linearly correlated with the size of novel genomes sequenced.  

***On what basis do the authors consider microbes the guardians of metabolism?***

Microbes are considered as the guardians of metabolism based on the fact of their ability to catalyze reactions in biogeochemical cycle and the central gene set for the ability distributed spatially and temporally. Microorganisms form Earth's environment like atmosphere through catalyzing redox reactions with enzymes translated from their genes. They using the same way control the biogeochemical cycles of nutrients and elements significant to life, such as nitrogen cycle and sulfate cycle. Although environmental selection may extinct a certain type of microbe, it is highly possible that the core gene responsible for a specific metabolic pathway in the microbial species survive in other cases. Eventually, the metabolic pathway is maintained over the long evolutionary time on Earth.   

#### References  
[1] P. G. Falkowski, T. Fenchel, E. F. Delong, "The Microbial Engines That Drive Earth’s Biogeochemical Cycles," *Science*, vol.320, pp. 1034-1039, May 2008.   
[2] T. Usui, I. Koike, and N. Ogura, N2O Production, Nitrification and Denitrification in an Estuarine Sediment. Estuarine, Coastal and Shelf Science, 2001,52: 769-781.  
[3] Suddick, E.C., Whitney, P., Townsend, A.R., The role of nitrogen in climate change and the impacts of nitrogen-climate interactions in the United States: foreword to thematic issue. Biogeochemistry, 2013, 114: 1-10.

## Module_01_Writing_Assignment_01

#### Prompt:  

*Microbial life can easily live without us; we, however, cannot survive without the global catalysis and environmental transformations it provides.” Do you agree or disagree with this statement? Answer the question using specific reference to your reading, discussions and content from evidence worksheets and problem sets.*  

#### Introduction  

Microbial engine have maintained the world for millions of years through global catalysis and environmental transformation. However things are changing in the new epoch. A new geological epoch name “Anthropocene” was proposed several years ago to describe human’s dominant impacts on Earth’s environment today [1]. Scientists have found a lot of current stratigraphic signatures caused by human disturbance distinct from that in the Holocene. Besides, technology and its exponential growth rate allow humans obtain the ability to alter the world at present or in the near future. There is even a saying that humans are able to survive without microbial support, because we can transform environments faster than microbes and control the direction of transformation to make environment more favorable to our life and activities. However, I still believe that humans can only survive in a world with microbes. One of the major reasons is that microbes are the engine of elements cycles and creates the environment sustaining our life. Secondly, there are still a plenty of biochemical pathways that we do not fully understand, making it difficult to replace microbes. And also, microbes can help us solve global issues we are encountering today like food scarcity and environmental pollution. I think they are our most reliable friends. 

#### Microbes can easily live without us	

There is no doubt that microbes can live on Earth easily without human beings, since they have been present on Earth for a longer time than humans and have huge abundance. Their ability to transfer genes horizontally and form protective cellular structure allow them survive unfavorable situations. 

Microbial history on Earth is an extraordinarily long story. At least 3.5 billion years ago, microbes appeared on Earth, and form Earth to what it looks like today [2]. However, as a mammal species, humans just started to stand on the stage of Earth around 200,000 years ago [3]. In addition, microbes are distributed almost everywhere on the planet, may be from marine subsurface about 15 km below the sea level to the atmosphere at an altitude of 57-77 km [4]. In contrast, the terrestrial surface of Earth is the major habitat for humans. Several factors help microbes exist on Earth over a vast span of evolutionary time. They are large species abundance, horizontal gene transformation and the specific cellular structure.   

Due to the large abundance and wide distribution of microbial species and genes, it is impossible to wipe out all of microbial species from our planet, as long as life continues to exist on earth. Microbial abundance is difficult to count. The number of prokaryotes on earth is roughly estimated to be 4-6×10^30^ cells[4], and microbial abundance in subseafloor sedimentary is reported to be 2.9×10^29^ cells [5]. Based on these figures with high magnitude, it is easily concluded that the number of microbes on the globe is extraordinarily huge. The large abundance enables microbes to mutate, acquire genetic diversity and form novel species at a high speed, and then easily adapt to new living conditions under environmental pressures [4][6]. One example is nitrogenase, an enzyme significant to nitrogen fixation and is irreversibly inhibited by oxygen. Nitrogen-fixing organisms have evolved mechanisms to protect nitrogenase from oxygen in order to survive in aerobic environment [6].  

What’s more, some central genes for biogeochemical cycles are horizontally transferred among microbial species, such as dissimilatory sulfite reductases and ammonia monooxygenases genes [6] for sulfate and nitrogen cycle respectively. As a result, even though a specific microbial species become extinct by negative natural selection, other alternative microbial species can take its place and occupy the ecological niche to keep elements cycles continuing. 

Besides, some microbial species are able to protect themselves from disadvantageous environments by forming a dormant cellular structure called endospore. This structure has high resistantance to detrimental living conditions like ultraviolet radiation, desiccation, high temperature and freezing, and can survive for a long period of time [7]. It is reported that endospores of Antarctic bacteria are capable of keeping dormant at a temperature of -14℃ for more than 10,000 years[8]. endospores also contain microbial genetic information. Thus it can revive itself to vegetative cells when environment becomes favorable. Bacillus and Clostridium are two examples of bacteria able to form endospore. This self-protective cellular structure may be helpful for microbes to endure several global mass-extinction episodes and keep alive over the millions of years. 

#### Human needs microbes

Human, however, is not able to live in a world without microbes. Microbes create a habitable living condition we are dependent on by catalyzing biogeochemical cycles and transforming Earth’s environment, while we are unable to mimic these biochemical processes on our own. Another reason is that microbes can help us to resolve some global issues we are facing today. 
  
*Microbes form our living environment*

Microbes form our living environment by making contributions in biogeochemical cycles and the evolution of Earth’s environment as well as constructing the inner environment for our bodies. 

Earth’s environment is affected by microbes through biogeochemical cycles. Six essential elements-H, C, N, O, S, and P- are the building blocks for all of living organisms on Earth, including human, to form their genetic information and cellular structures. The global element cycles largely driven by microbially catalyzed redox reactions, introduce these elements into biosphere from abiotic environment [6]. For example, nitrogenase, an enzyme produced by microbes, catalyzes the redox reaction converting nitrogen gas in atmosphere into ammonium which is then oxidized to nitrate and absorbed by plants and animals through food chains. Although there are non-biological processes like lightning able to fix nitrogen, the biological process described above is the predominant pathway. Therefore, these six essential elements might not be available for humans, if the microbial links between abiotic environment and biosphere in the global cycles were broken. 

Microbes also play significant roles in the evolution of Earth’s environment. The evolution of the structure of microbial mats might result in the emergence of photosynthesis around 3.5 billion years ago, which equipped atmosphere with oxidation power, increased microbial productivity, and allowed organisms to live freely on the planet [3]. After 1.3 billion years, green bacteria with chloroplasts working with other microbial species increased atmospheric oxygen level drastically [3]. Besides, microbes make substantial contributions to the formation of modern atmosphere. Most of oxygen we breathe today is not from terrestrial photosynthesis, but from marine photosynthesis performed mostly by microorganisms like eukaryotic algae and cyanobacteria [9]. Thus, it can be imagined that humans might die due to shortage of atmospheric oxygen if microbes stopped producing oxygen.  

In addition to impacting the external environment, microbes also have effects on our health. The relationship between some microbial species and humans is mutualism, which means we cannot life healthily without these microorganisms. For instance, we get benefits from gastrointestinal microbiota fermenting dietary fiber into short-chain fatty acids, which are then utilized as energy and nutrients and oxidized to carbon dioxide through respiration process in our bodies [10]. This microbial community also helps us to adsorb essential minerals like calcium and iron and synthesize some important compounds, such as vitamin B and xenobiotics. Therefore we might not be able to gain sufficient energy and nutrients to sustain our life without gastrointestinal microbiota.    

*Human technology cannot replace microbes*

Although scientific technology has developed rapidly over the past few decades and will continue to grow at an unprecedented rate, we will still rely on the environmental conditions provided by microbes in the foreseeable future. The main reason is the hardness to comprehend all microbial metabolisms and replace microbial functions in biogeochemical cycles due to microbes’ high mutation rate and lack of sufficient information. 

It is extremely difficult for humans to fully understand, manipulate or even mimic all microbial metabolisms because of their complexity and diversity. Until now, the biological processes for nitrogen cycle are still being investigated. New microbial species, such as ammonia-oxidizing archaea [11] and comammox [12], are discovered recently and revise our understanding on nitrogen cycle. But their relationships with ammonia-oxidizing bacteria and nitrite-oxidizing bacteria are still a mystery. Additionally, it is estimated that there about one trillion species of microbes on Earth, while 99.999 percent of them have not been discovered [13]. As a result, it is highly possible that there are some species that are of importance in biogeochemical cycles but have not been discovered by humans.

The high mutation rate of microbial genes is another factor that obstructs our comprehension on microbial mechanisms. With the assumption of a prokaryotic mutation rate of 4×10^-7^ mutations per gene per DNA replication, Whitman et al. calculated four simultaneous mutation in every gene shared by the population of marine heterotrophs and soil prokaryotes at high rate of once every 0.4 and 3.4 hours [4]. The high mutation rate leads to the high-speed changes of genetic diversity, and consequently novel mechanisms are created that are difficult to detect at the first time and first place.

Lack of comprehensive information on microbes, it is really hard for us to perform biochemical mechanisms artificially and replace microbes in the global biogeochemical cycles. And also we do not have sufficient knowledge on the intricate relationships among microbial species and between microbes and abiotic environment. We might underestimate the significance of some mechanisms to the global environment. As a result, the likelihood to generate a favorable condition for human life by ourselves is very small, even if technology is advanced enough to allow humans to mimic microbial mechanisms.

*We still need microbes to solve global issues*

Microbes not only form a habitable environment for humans, but also play critical roles in human activities and help us address global issues like famine and environmental pollution. For instance, we develop genetic engineering technology to form new cells with targeting genes which are able to generate certain proteins and applied in a variety of industries like food production and diseases treatment. This technology need to use microbial cells to achieve gene-engineered bacteria. Another example is wastewater treatment. In order to prevent eutrophication in aquatic environment, bacteria involved in nitrogen and phosphorus cycles, such as ammonium oxidizing bacteria and polyphosphate-accumulating organisms, are employed in wastewater treatment processes to remove nitrogen and phosphate from the effluent [14].  

#### Conclusions

Thanks to microbes for providing such a wonderful place with abundant nutrients and perfect atmosphere for us, and helping us resolve our social, economic and environmental issues. They are amazing organisms with simple cellular structures compared with human, but too complex to been completely understood. It is hard to imagine a world without microbes. But one thing is definite that human cannot survive in such a world. Since we are so relied on these powerful organisms, we should cooperate with them to mitigate our negative impacts on Earth which is the home not just for us but also for every living species.     

#### References

[1] C. N. Waters, J.Zalasiewicz, C. Summerhayes, “The Anthropocene is functionally and stratigraphically distinct from the Holocene,” *Science*, vol.351, issue 6269, pp. 2622/1-2622/10, Jan. 2016.  
[2] E.G. Nisbet and N.H. Sleep, “The habitat and nature of early life,” *Nature*, vol.409, pp. 1083-1091, Feb. 2001.   
[3] D. P. Schrag, “Geobiology of the anthropocene,” in *Fundamentals of Geobiology*, 1st edition, Blackwell Publishing Ltd., 2012, pp. 425-436.  
[4] W. B. Whitman, D. C. Coleman, and W. J. Wiebe, “Prokaryotes: The unseen majority,” *Pro. Natl. Acad. Sci.*, vol. 95, pp. 6578-6583, June 1998.   
[5] J. Kallmeyer, R. Pockalny, and R. R. Adhikari, “Global distribution of microbial abundance and biomass in subseafloor sediment,” *PNAS*, pp. 1-4.  
[6] P. G. Falkowski, T. Fenchel, and E. F. Delong, “The microbial engines that drive earth’s biogeochemical cycles,” *Science*, vol.320, pp. 1034-1039, May 2008.    
[7] *Endospore* (n.d.) [Online]. Available: https://en.wikipedia.org/wiki/Endospore   
[8] J. G. Black, “Characteristics of Prokaryotic and Eukaryotic Cells,” in *Microbiology: principles and explorations*, 7th edition, Wiley, 2011, pp. 90-93.    
[9] J. F. Kasting and J. L. Siefert, “Life and the evolution of earth’s atmosphere,” *Science*, vol.296, pp. 1066-1068, May 2002.   
[10] *Gut flora* (n.d.) [Online]. Available: https://en.wikipedia.org/wiki/Gut_flora#Functions   
[11] R. Hatzenpichler, “Diversity, physiology, and niche differentiation of ammonia-oxidizing archaea,” *Applied Environmental Microbiology*, vol.78, pp. 7501-7510, Nov.2012.   
[12] D. Holger, E.V. Lebedeva, P. Pjevac, P. Han, C. Herbold, M. Albertsen, N. Jehmlich, M. Palatinszky, J. Vierheilig, A. Bulaev, R. H. Kirkegaard, M. von Bergen, T. Rattei, B. Bendinger, P.H. Nielsen, M. Wanger “Complete nitrification by Nitrospira bacteria,” *Nature*, doi:10.1038/nature16461.  
[13] N. Bakalar. (2016, May 23). *Earth May Be Home to a Trillion Species of Microbes* [Online]. Available:  https://www.nytimes.com/2016/05/24/science/one-trillion-microbes-on-earth.html   
[14] Metcalf and Eddy, *Wastewater engineering treatment and reuse*, 4th Edition, McGraw Hill, 2003.

## Module_01_Additional_Readings

[1] J. Kallmeyer, R. Pockalny, R. R. Adhikari, D. C. Smith, and S. D'Hondt, "Global distribution of microbial abundance and biomass in subseafloor sediment," *PNAS*, pp: 1-4, July 2012. (http://www.pnas.org/content/109/40/16213)

[2] R. A. Welch, V. Burland, G. Plunkett III, P. Redford, P. Roesch, D. Rasko, E. L. Buckles, S.-R. Liou, A. Boutin,
J. Hackett, D. Stroud, G. F. Mayhew, D. J. Rose, S. Zhou, D. C. Schwartz, N. T. Perna, H. L. T. Mobley, M. S. Donnenberg, and F. R. Blattner, "Extensive mosaic structure revealed by the complete genome sequence of uropathogenic *Escherichia coli*," *PNAS*, vol. 99, no. 26, pp:17020-17024, Dec. 2002. (http://www.pnas.org/content/99/26/17020.long)

[3] J. Achenbach. (2012, Jan. 2). *Spaceship Earth: A new view of environmentalism* [Online]. Available: https://www.washingtonpost.com/national/health-science/spaceship-earth-a-new-view-of-environmentalism/2011/12/29/gIQAZhH6WP_story.html?utm_term=baff32d71704.

[4] D. P. Schrag, "Geobiology of the Anthropocene," in A. H. Knoll, D. E. Canfield, and K. O. Konhauser (Eds), *Fundamentals of Geobiology*, 1st edition, Blackwell Publishing Ltd., 2012, pp. 425-436. (https://onlinelibrary.wiley.com/doi/10.1002/9781118280874.ch22)

[5] C. Mooney. (2016, Jan. 7). *Scientists say humans have now brought on an entirely new geologic epoch* [Online]. Available: https://www.washingtonpost.com/news/energy-environment/wp/2016/01/07/scientists-say-humans-have-now-brought-on-an-entirely-new-geologic-epoch/?utm_term=323fc2cc7322.

[6] P. Falkowski, R. J. Scholes, E. Boyle, J. Canadell, D. CanÞeld, J. Elser, N. Gruber, K. Hibbard, P. Ho¬gberg, S. Linder, F. T. Mackenzie, B. Moore III, T. Pedersen, Y. Rosenthal, S. Seitzinger, V. Smetacek, W. Steffen, "The global carbon cycle: A test of our knowledge of earth as a system," *Science*, vol.290, no. 13, pp. 291-296, Oct. 2000. (http://science.sciencemag.org/content/290/5490/291)

[7] A. Leopold, "The Land Ethic," in *A Sand Country Alamanac*. (http://rintintin.colorado.edu/~vancecd/phil3140/Leopold.pdf)

[8] D. E. Canfield, A. N. Glazer, P. G. Falkowski, "The evolution and future of earth's nitrogen cycle," *Science*, vol.330, pp. 192-196, Oct. 2010. (http://science.sciencemag.org/content/330/6001/192.long)

[9] Zehnder, A. J. B., & Stumm, W. (1988). Geochemistry and Biogeochemistry of anaerobic habitats. In A. J. B. Zehnder (Ed.), *Biology of anaerobic microorganisms*. New York: Wiley. (https://www.dora.lib4ri.ch/eawag/islandora/object/eawag:11376)

[10] E. G. Nisbet and N. H. Sleep, "The habitat and nature of early life," *Nature*, vol. 409, pp. 1083-1091, Feb. 2001. (https://www.nature.com/articles/35059210)

[11] W. B. Whitman, D. C. Coleman, and W. J. Wiebe, "Prokaryotes: The unseen majority," *PNAS*, vol. 95, pp. 6578-6583, June 1998. (http://www.pnas.org/content/95/12/6578)

[12] C. N. Waters, J. Zalasiewicz, C. Summerhayes, A. D. Barnosky,
C. Poirier, A. Gałuszka, A. Cearreta, M. Edgeworth, E. C. Ellis, M. Ellis, C. Jeandel, R. Leinfelder, J. R. McNeill, D. deB. Richter, W. Steffen, J. Syvitski, D. Vidas, M. Wagreich, M. Williams, Z. An, J. Grinevald, E. Odada, N. Oreskes, A. P. Wolfe, "The Anthropocene is functionally and stratigraphically distinct from the Holocene," *Science*, vol.351, issue 6269, pp. 137, Jan. 2016. (http://science.sciencemag.org/content/351/6269/aad2622)

[13] P. G. Falkowski, T. Fenchel, E. F. Delong, "The Microbial Engines That Drive Earth’s Biogeochemical Cycles," *Science*, vol.320, pp. 1034-1039, May 2008. (http://science.sciencemag.org/content/320/5879/1034.full)

[14] J. RockstrÖm, "A safe operating space for humanity," *Nature*, vol. 461, pp. 472-475, Sep. 2009. (https://www.nature.com/articles/461472a)

[15] J. F. Kasting and J. L. Siefert, "Life and the Evolution of Earth's Atmosphere," *Science*, vol. 296, pp. 1066-1068, May 2002. (http://science.sciencemag.org/content/296/5570/1066) 

# Module_02: Remapping the body of the world

## Module_02_Evidence_Worksheet_04: “Bacterial Rhodopsin Gene Expression”

#### Reading:

A. Martinez, A. S. Bradley, J. R. Waldbauer, R. E. Summons, and E. F. DeLong, "Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host," *PNAS*, vol. 104, no. 13, pp. 5590-5595, Mar. 2007.

####Learning objectives:  

•	Discuss the relationship between microbial community structure and metabolic diversity  
•	Evaluate common methods for studying the diversity of microbial communities  
•	Recognize basic design elements in metagenomic workflows

#### General Questions:  

***What were the main questions being asked?***

1.	What are the various specific functions and physiological roles of proteorhodopsins (PRs) photosystem?
2.	What are the effects of PR-catalyzed proton translocation on photophosphorylation in a heterologous host? 

***What were the primary methodological approaches used?***

First of all, a fosmid library from a planktonic sample containing PR clones was screened on retinal-containing LB agar plating medium. Potential PR-expressing clones were identified based on their visual appearance of orange or red color on the medium. Next, fosmid DNA of the potential clones was transformed into *E. coli* EPI300 and analyzed by sequencing a collection of transposon-insertion clones with primers T7 and EpiFos5R, and the DNA sequences were studied based on ORF content similarity to homologues in NCBI database.

In order to obtain evidence for the functional annotations of PR photosystem genes, different transposon mutant phenotypes with an insertion in predicted genes were analyzed using EPI300 cells as hosts on LB plates containing chloramphenical, kanamycin, and 0.02% L-arabinose. Pigment in different mutants was analyzed by high-performance liquid chromatography (HPLC). Besides, light-activated proton-translocation activity was studied based on light-dependent change in pH to analyze PR expression of the potential fosmids.  

Photophosphorylation in E.coli caused by PR-driven proton translocation was investigated by comparing light-induced changes in ATP levels in the clones containing PR photosystem and PR- mutant derivatives. The effects of carbonylcyanide m-chlorophenylhydrazone (CCCP) and N,N’-dicyclohexylcarbodiimide (DCCD) on PR^+^ *E.coli* cells were tested to evaluate light-driven photophosphorylation more comprehensively. ATP was measured using a luciferase-based assay. 
 
***Summarize the main results or findings.***

Two clones, HF10_19P19 and HF10_25F10, were detected as potential PR-expressing clones on the LB agar plating medium when high copy number was induced. The clones had high identity and similarity to other BAC clones containing PR from Alphaproteobacteria. The two potential PR-expressing clones studied shared similar genes on PR and retinal biosynthesis operon, which had been observed in other PR-containing clones. 

The study of transposon-insertion mutants showed that HF10_19P19 contained all genes required for retinal biosynthesis in *E.coli*. PR-associated predicted retinal biosynthetic pathway was verified, which involves genes *idi*, *ispA*, *crtE*, *crtB*, *crtI*, *crtY*, and *blh*. However, it was observed that gene *idi* had already presented in *E.coli* before transformation of the fosmid vector. The two clones detected in the study were able to independently express a functional PR with light-activated proton-translocating activity. 

Light-activated, PR-induced proton motive force was proved to be able to drive ATP synthesis. Similarly, PR photosystem genes from the two studied clones were necessary and sufficient to drive photophosphorylation in *E.coli* cells. Additionally, H^+^-ATP synthase was responsible for the increase in the light-activated ATP.

***Do new questions arise from the results?***

1.	What was the cause of the low levels of retinal in the carotenoid extraction from the transposon insertion mutants? Was that the polar effects led by the transposon insertion in downstream expression, or pathway inhibition because of product accumulation?

2.	How to detect PR-containing clones when the fosmid vector was induced to low copy number? 

3.	How to interpret the negative results on undetectable light-stimulated growth rates or yields in PR-containing bacteria in seawater in some studies?

***Were there any specific challenges or advantages in understanding the paper (e.g. did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?***

This paper explained the methodology it used to achieve its objectives logically and adequately. Reasons behind methodology were given in the paper and corresponding expectation was also provided to explain the results. For instance, in the study of fosmid library screening, the reasons of clones with orange or red phenotype being considered as potential PR-expressing clones was demonstrated by the phenomenon that retinal enables cells to acquire a red or orange pigmentation. In addition, the reasons of the test on photophosphorylation in cells caused by PR-driven proton translocation were the previous hypothesis analogous to earlier studies of haloarchaeal bacteriorhodopsins. 

Conclusions were strongly supported by evidence observed in this research. One of conclusions that the potential PR-expressing clones contained all genes required for retinal biosynthesis was supported by cell pigmentation as well as HPLC pigment analysis which indicated pigment intermediate accumulated in different transposon-insertion mutants. 

Figures in the paper WERE easy to understand. The flow chart of retinal biosynthesis pathway clearly illustrated functions of each retinal biosynthesis genes. The mechanics of light-activated, PR-enabled photophosphorylation in E.coli was easy to comprehend by a figure as well. 

## Module_02_Problem_set_03: ”Metagenomics: Genomic Analysis of Microbial Communities“

#### Learning objectives:  

Specific emphasis should be placed on the process used to find the answer. Be as comprehensive as possible e.g. provide URLs for web sources, literature citations, etc.

#### Reading:

Wooley JC, Godzik A, Friedberg I. 2010. A primer on metagenomics. PLoS Computational Biology. Vol. 6, issue 2: e1000667.

#### Specific Questions:

***How many prokaryotic divisions have been described and how many have no cultured representatives (microbial dark matter)?***

By 2016, we have obtained information on 89 bacterial phyla and 20 archaeal phyla via small 16S RNA database. However, it could be up to 1500 bacteria phyla, since some microbes live in the “shadow biosphere” and need to be found in the future [1]. 

26 of the approximately 52 identifiable major phyla within the domain bacteria have cultivated representatives by 2003 [2]. But the proportion may increase over the past decade.   

***How many metagenome sequencing projects are currently available in the public domain and what types of environments are they sourced from?***

Until now, there are at least about 110,000 metagenome sequencing projects available in European Bioinformatics Institute database. The number is always changing. These projects are from all environment types, including sediments, soil, and aquatic environments.    

***What types of on-line resources are available for warehousing and/or analyzing environmental sequence information (provide names, URLS and applications)?***

1. Kyoto Encyclopedia of Genes and Genomes (KEGG): http://www.genome.jp/kegg/   
KEGG is a database resource for genomes, biological pathways, diseases, drugs, and chemical substances. It can be applied in bioinformatics research and education, data analysis in genomics, metagenomics, metabolomics, modeling and simulation in systems biology, and translational research in drug development. 

2. MEtaGenome Analyzer (MEGAN): http://ab.inf.uni-tuebingen.de/software/megan6/  
MEGAN is a comprehensive analyzing tool for microbial data. It can be employed for taxonomic analysis using the NCBI taxonomy or a customized taxonomy such as SILVA, functional analysis using InterPro2GO, SEED, eggNOG or KEGG, bar charts, word clouds, Voronoi tree maps and many other charts, PCoA, clustering and networks, metadata supports, and many different types of input

3. Integrated Microbial Genomes & Microbiomes (IMG/M): https://img.jgi.doe.gov/cgi-bin/m/main.cgi  
This system serves as a community resource for analyzing and annotating genome and metagenome datasets in comprehensive comparative context. The IMG data warehouse integrates genome and metagenome datasets provided by IMG uses with a comprehensive set of publicly available genome and metagenome datasets. There is a set of tools like IMG UI Map available in the system for comparative analysis of microbial genomes. A genome annotation pipeline is provided by IMG/G to integrates information from several tools, including KEGG, Pfam, InterPro, and the Gene Ontology. 

4. Metagenomic Rapid Annotations using Subsystems Technology (MG/RAST): https://www.mg-rast.org/   
The RAST technology was originally performed to allow automated high-quality annotation of complete or draft microbial genomes using SEED data and has been adapted for metagenome analysis. The annotation of sequence fragments is provided by the server, including phylogenetic classification, functional classification of samples, and comparison between multiple metagenomes. The server is able to compute an initial metabolic reconstruction for the metagenome and allow comparison of metabolic reconstructions of metagenomes and genomes.

5. National Center for Biotechnology Information (NCBI): https://www.ncbi.nlm.nih.gov/   
NCBI provides databases for genome sequences, functional genomics, protein sequences, and functional domains, and bioinformatics tools for analysis. GenBank for DNA sequences and PubMed, a bibliographic database for the biomedical literature, are two of major databases. Genome Blast sequence can be used for searching genome sequences.

6. European Bioinformatics Institute (EBI): https://www.ebi.ac.uk/   
EBI provides data resources on genomes and metagenomes, like Ensembl for genome and ChEMBL for binding, functional and ADMET bioactivity. Tools for analysis and search genomes and metagenome are also available in its website.  

7. OMICTools: https://omictools.com/   
OMICtools is a community-driven search engine for biological data analysis. It offers an easy means of locating the right tools corresponding to each researcher and their specific biological data analyses. The OMICtools website centralizes more than 18,500 software tools and databases, manually classified, by a team of biocurators including many scientific experts. Key information, a direct link, and access to discussions and evaluations by the biomedical community are provided for each tool. Researchers can directly upload their code versions to improve research traceability [3]. 

8. SILVA: https://www.arb-silva.de/   
SILVA provides abundance information for metagenomics study, including fully aligned and up to date small (16S/18S, SSU) and large (23S/28S, LSU) subunit ribosomal RNA “Parc” databases on the webpage and ARB files, preconfigured subsets of high-quality, full-length sequences as ARB and FASTA files, extensive search for sequence retrival, a clear rating system for all steps for data processing with emphasis on sequence and alignment quality, and substantial support related to all aspects of data sets, sequence analysis and probe/primer design. 

9. AmpliconNoise: https://code.google.com/archive/p/ampliconnoise/   
AmpliconNoise is a collection of programs for the removal of noise from 454 sequenced PCR amplicons. Two steps are involved, the removal of noise from the sequencing itself and the removal of PCR point errors. This project also includes the Perseus algorithm for chimera removal.

10. Ribosomal database project (RDP): http://rdp.cme.msu.edu/   
RDP provides quality-controlled, aligned and annotated bacterial and archaeal 16S rRNA sequences, and Fungal 28S rRNA sequences, and a suite of analysis tools to the scientific community like classifier, probe match, and RDPipeline. 

***What is the difference between phylogenetic and functional gene anchors and how can they be used in metagenome analysis?***

Phylogenetic and functional gene anchors have different patterns in gene transfer. Phylogenetic anchors are transferred vertically, while functional anchors transfer gene horizontally. Phylogenetic gene anchors are ideally single-copy and carry phylogenetic information which allows tree reconstruction. Functional anchors, however, are utilized for identification of specific biogeochemical functions and not useful for phylogeny.  

Phylogenetic anchors are applied in sequence–based analysis, in which phylogenetic anchors can be found in the flanking DNA to provide a link between phylogeny and the functional gene when a gene of interest is identified. In contrast, functional gene anchors are employed in function-based analysis. Phylogenetic anchors and genomic structure in the flanking DNA are identified by sequencing a collection of clones with a common function. This analysis can provide a slice of the metagenome that determin the diversity of genomes with a specific function [4]. 

***What is metagenomic sequence binning? What types of algorithmic approaches are used to produce sequence bins? What are some risks and opportunities associated with using sequence bins for metabolic reconstruction of uncultivated microorganisms?***

Sequence binning is the process that associates sequence data with its original OUT. Sequences are mapped onto genome using pre-existing genome database of similar organisms or grouped based on different qualities.   
There are two approaches for sequence binning. One is composition-based binning. The other is similarity-based binning. Composition-based binning applies Markov models based on k-mer frequencies to classify sequences. This binning strategy includes TETRA, PhyloPythia, GSOM, and codon-usage methods. Similarity-based binning utilizes similarity function sim to find similarities between samples and reference sequences database to build a species tree. 

One of major problems of the application of composition-based binning is that if the OTUs are closer and numerous, the frequency of misclassification errors are high. Additionally, incomplete coverage of reference genome database may miss key parts and become less representative. However, k-mer is still a promising tool for binning ORFan sequences with few or no homologs, since it does not require for reference sequences. 

Similarity-based binning is implemented by some biotechnology analysis software like MEGAN and CARMA. The species trees produced via the approach is able to show an overview of the dominant species in the sample. But in some cases, if genome from other organisms have similar properties, it may be impossible to produce a precise assignment to an OTU. 

***Is there an alternative to metagenomic shotgun sequencing that can be used to access the metabolic potential of uncultivated microorganisms? What are some risks and opportunities associated with this alternative?***

There are several alternatives to shotgun sequencing: functional metagenomics, single-cell sequencing, fluorescence in situ hybridization (FISH), second generation sequencing, third generation sequencing, and polony-based methods. Their risks and opportunities are discussed as below. 

Functional metagenomics based on phenotypic screening is applied for the study of functions of proteins and other gene produces by DNA sequences [5]. This approach is potential to overcome the limitation of the absence of most microbial diversity from pure culture and discover more novel gene products when DNA sequence is not predictive. However, the limitations of functional metagenomics include cloning bias in library construction, range restrictions on library vector host, and gene expression dependent on host [5]. 

Single-cell sequencing provides the genomic information of an individual cell using advancing sequencing methodology [6]. It has been proved to be a powerful method to obtain new genomes of uncultivated microorganisms. A large number of taxonomic groups have their reference genome published with the application of single-cell sequencing [6]. Nevertheless, this technology may produce inaccuracy data due to amplified sequence bias, genomic dropouts and genetic material contamination [7].

Fluorescence in situ hybridization (FISH) has been widely used for identification of the taxonomy and potential functions of microbes [8]. It is an imaging technique using fluorescent probe to bind with specific sequence like rRNA in cell. This technique can be applied in combination with single-cell isolation techniques for sequencing the whole genome of uncultivated microorganisms [8]. Recent development in this approach, such as individual genes-FISH and catalyzed reporter deposition-FISH, allow researchers to find functional genes with low abundance [8]. 

Second generation sequencing methods have been developed and replace shotgun sequencing for small sized genomes and environmental genomics [9]. These methods are based on the generation of “polymerase colonies”, which enable enzymatic reactions to perform concurrently with the sequence of nucleic acid [9].

 Third generation sequencing is developed to reduce or eliminate the main challenges in sequencing technology like short read lengths and significant error rate [10]. This technology enables us to reconstruct the genome, comprehend genomic variations, recover transcriptomes, and indentify the transcription factor binding sites [11].
 
The number of sequences produced by polony-based methods is larger than shotgun sequencing, but those sequences are shorter than that generated by the first generation sequencing [9]. Although this alternative is unable to read long mononucleotide repeats correctly, it is feasible to sequence whole genomes by sheer volume of DNA sequenced [9]. 

#### References  
[1] Solden L, Lloyd K, Wrighton K. 2016. The bright side of microbial dark matter: lessons learned from the uncultivated majority. Environmental Microbiology. Vol. 31: 217-226  
[2] Rappe MS, Giovannoni SJ. 2003. The uncultured microbial majority. Annu. Rev. Microbiol. 57: 369–394.  
[3] Perrin H, Denorme M, Grosjean J, Dynomant E, Henry VJ, Pichon F, Darmoni S, Desfeux A, Gonzalez BJ. 2017. OMICtools: a community-driven search engine for biological data analysis.  Link: https://arxiv.org/ftp/arxiv/papers/1707/1707.03659.pdf.   
[4] Srivastava S, Ghosh N, Pal G. 2013. Chapter 10: metagenomics: mining environmental geomes, p 161-190. In Kuhad RC, Singh A (ed), Biotechnology for Environmental Management and Resource Recovery. Springer, New York, NY. DOI: 10.1007/978-881-322-0876-1.   
[5] Cheng J, Lam KN, Engel K, Hall M, Neufeld JD, Charles TC. 2017. Chapter 1: metagenomic cosmid libraries suitable for functional screening in proteobacteria, p 1-12. In Charles TC, Liles MR, Sessitsch A (ed), Functional Metagenomics: Tools and Applications. Springer, New York, NY. DOI: 10.1007/978-3-319-61510-3.   
[6] Chen Y, Wang F. 2015. Chapter 3: single-cell sequencing of microorganisms, p 31-40. In Wang X (ed), Single Cell Sequencing and Systems Immunology. Springer, New York, NY. DOI:10.1007/978-94-017-9753-5.  
[7] Li B, Zhang X, Zheng J. 2015. Single-cell next-generation sequencing and its applications in cancer biology, p 1-18. In Wu W, Choudhry H (ed), Next Generation Sequencing in Cancer Research, Volume 2: From Basepairs to Bedsides. Springer, New York, NY. DOI: 10.1007/978-3-319-15811-2.  
[8] Ikarm M, Malik A. 2013. Recent development in the methods of studying microbial diversity p 17-51. In Malik A, Grohmann E, Alves M (ed), Management of Microbial Resources in the Environment. Springer, New York, NY. DOI: 10.1007/978-94-007-5931-2.   
[9] Wooley JC, Godzik A, Friedberg I. 2010. A primer on metagenomics. PLoS Computational Biology. Vol. 6, issue 2: e1000667.   
[10] Chapter 6: New horizons in next-generation sequencing, p 51-57. In Metwally SE, Ouda OM, Helmy M (ed), Next Generation Sequencing Technologies and Challenges in Sequence Assembly.2014. Springer, New York, NY. DOI: 10.1007/978-1-4939-0715-1.  
[11] Sung WK (ed). 2017. Algorithms for Next-Generation Sequencing. CRC Press, Taylor & Francis Group, LLC, Boca Raton, FL.  

## Module_02_Additional_Readings

[1] J. C. Wooley, A. Godzik, I. Friedberg, "A primer on metagenomics," *PLOS Computational Biology*, vol. 6, issue 2, pp. 1-13, Feb. 2010. (http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000667)

[2] M. Taupp, K. Mewis, and S. J. Hallam, "The art and design of functional metagenomic screens," *Current Opinion in Biotechnology*, 22: 1-8, 2011. (https://www.sciencedirect.com/science/article/pii/S0958166911000371?via%3Dihub)

[3] E. L. Madsen, "Identifying microorganisms responsible for ecologically significant biogeochemical processes," *Nature*, vol. 3, pp. 439-446, May 2005. (http://www.nature.com/articles/nrmicro1151)

[4] A. Martinez, A. S. Bradley, J. R. Waldbauer, R. E. Summons, and E. F. DeLong, "Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host," *PNAS*, vol. 104, no. 13, pp. 5590-5595, Mar. 2007. (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838496/)

# Module_03: Microbial Sepcies Concepts

## Module_03_Evidence_Worksheet_05: “Extensive mosaic structure”

#### Readings;

R. A. Welch, V. Burland, G. Plunkett, P. Redford, P. Roesch, D. Rasko, E. L. Buckles, S.-R. Liou, A. Boutin, J. Hackett, D. Stroud, G. F. Mayhew, D. J. Rose, S. Zhou, D. C. Schwartz, N. T. Perna, H. L. T. Mobley, M. S. Donnenberg, and F. R. Blattner, "Extensive mosaic structure revealed by the complete genome sequence of uropathogenic Escherichia coli," *PNAS*, vol. 99, no. 26, pp. 17020-17024, Dec. 2002.

#### Part 1: Learning objectives:

*	Evaluate the concept of microbial species based on environmental surveys and cultivation studies.

*	Explain the relationship between microdiversity, genomic diversity and metabolic potential

*	Comment on the forces mediating divergence and cohesion in natural microbial communities

#### General Questions: 

***What were the main questions being asked?***

What are the genetic bases for pathogenicity and the evolutionary diversity of *E.coli*?

***What were the primary methodological approaches used?***

*E.coli* CFT073 was firstly cloned and sequenced by using dye-terminator chemistry and data was collected on Applied Biosystem ABI377 and 3700 automated sequencers. SEQMANII was applied for sequence data assembly. And then, the genome sequence was annotated by MAGPIE using GLIMMER to define ORFs. Orthology was inferred if matches for CFT073 genes with either MG1655 or EDL933 genes were equal to or larger than 90% identity. Genome comparisons were performed using a modified method employed for the comparison of EDL933 with MG1655.  

***Summarize the main results or findings.***

The complete genome sequence of Uropathogenic *E.coli* CFT073 was a circular, 5,231,428-bp chromosomal sequence, containing five cryptic prophage genomes but not any virulence plasmid. A three-way genome comparison of CFT073, EDL933, and MG1655 indicated that only 39.2% of their combined sets of proteins were shared to all three strains. 

The codon usage patterns had no significant difference in backbone between CFT073 and EDL933, while most of them in CFT073 backbone had distinguishable difference from those in island ORFs. Although islands in CFT073 contained 2004 genes, only 204 of them were shared with EDL933. Comparison between CFT073 and EDL933 revealed that many islands with length larger than 4kb were located at the same relative backbone position in the two *E.coli* although islands contents were not related. 

The distinctions in disease potential between enterohemorhaggic and uropathogenic *E.coli* were that CFT073 did not contain genes for type III secretion system and virulence genes encoded by phage and plasmid which were common to *E. coli* O157:H7.

A number of possible factors had been reflected in CFT073 genome sequence might have contributions to colonization of the urinary tract tissues and the disease. Genes had been found in CFT073 for encoding putative fimbriae adhesins, autotransporters, iron-sequestration systems, and phase-switch recombinases.   

The common core chromosome in different *E.coli* pathotypes had been maintained throughout its vertical evolution, while many islands interrupting the common backbone were obtained through different horizontal transfers in each strain. 

***Do new questions arise from the results?***

1.	How to design far more discriminating tools for diagnosis of particular *E.coli* pathotypes that cause such a wide range of intestinal and extraintestinal diseases?

***Were there any specific challenges or advantages in understanding the paper (e.g. did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)? ***

One of advantages of the paper was sufficient background information provided, making it easy for readers to understand the research’s objectives and significance. The authors discussed distinct genotypes of *E.coli* harmful to human health and their negative effects on human body, which enabled readers to understand why the research was carried out. 

Conclusions in the paper were strongly supported by findings. For instance, in order to substantiate potential of CFT073 for new niches and different pathogenic mechanisms, genes encoding fimbriae adhesions, autotransporters, and cytolytic toxin were investigated. 

Figures in the paper effectively illustrated comparison of CFT073 with other *E.coli* strains. The figures presenting locations and sizes of CFT073 and EDL933 islands not only supported author’s findings, but also allowed readers to compare the two strains by themselves.

A challenge when reading the paper is that results and corresponding figures are split far away in the text, which increases difficulty to associate figures with their analysis. For example, the discussion on islands and horizontal transfer is in the second page, while its corresponding figure showing the locations and sizes of CFT073 and EDL933 islands is in the fourth page.  Therefore, modification on text layout might improve convenience for reading. 

#### Part 2: Learning Obejectives:

•	Comment on the creative tension between gene loss, duplication and acquisition as it relates to microbial genome evolution

•	Identify common molecular signatures used to infer genomic identity and cohesion

•	Differentiate between mobile elements and different modes of gene transfer

***Based on your reading and discussion notes, explain the meaning and content of the following figure derived from the comparative genomic analysis of three E. coli genomes by Welch et al. Remember that CFT073 is a uropathogenic strain and that EDL933 is an enterohemorrhagic strain. Explain how this study relates to your understanding of ecotype diversity. Provide a definition of ecotype in the context of the human body. Explain why certain subsets of genes in CFT073 provide adaptive traits under your ecological model and speculate on their mode of vertical descent or gene transfer.***

![](Module_03_Worksheet_05.png)

This figure shows locations and sizes of CFT073 and EDL933 islands with length over 4 kb. The vertical axis is island size, and horizontal axis is island position in collinear backbone. Islands are indicated by tRNA labels and one insertion target tmRNA (*ssrA*) label. * is the label showing islands in the two strains locate at the same backbone position but not near tRNAs. This figure reveals that nine islands are at the same tRNA in both genomes, and that ten other locations are occupied by unrelated islands in the two strains. 

This study demonstrates that ecotype diversity within a species is the result of genotypic adaptation to specific environmental conditions. Due to islands gene and horizontal gene transmission, different ecotypes may share low DNA and amino acid sequence identity with each other. In this study, CFT073 and EDL933 are distinct ecotypes of *E.coli* and only have 39.2% shared *E.coli* proteins. The rest distinct proteins allow CFT073 and EDL933 to adapt to different habitats-urinary tract and intestine respectively-in human body, On the other hand, the results in this study shows that ecotypes are not different species although they have distinguishable phenotypes resulting from different genotypes. Ecotypes can potentially interbreed and possess common genomes that determine essential characteristics of the species like nutrient synthesis pathways. As ecotypes of *E.coli*, CFT073 and EDL933 share a set of backbone genes with high identity (>98%) and have a shared codon bias, which means they are different types of *E.coli* colonizing different sites in human body.    

Ecotypes are variants of a species that present different phenotypes when they colonize different parts of human body, while the phenotypic differences are too subtle to be considered as a new species.

CFT073 is an uropathogenic strain harmful when it enters urinary tract, bloodstream or cerebrospinal fluid. Its specific colonization may be the result of some certain island genes. The discovery of genes encoding 12 putative fimbriae and five different copies of genes similar to *fimBE* in CFT073 genomes, reflects that CFT073 may have specific surface structures allow CFT073 to attach to host cells so that its colonization in urinary tract is consolidated. Additionally, Sat, having cytopathic effects on bladder and kidney cells, is secreted from CFT073 cells by autotransporters encoded by specific DNA sequence in CFT073 genomes.  As a result, these special genes allow CFT073 to colonize urinary tract and other certain parts of human body, and thus lead to diseases. 

CFT073 may vertically transfer their genes to the next generation through DNA replication and binary fission, the most common reproduction approach for bacteria. In addition to the genes obtained from parental cells, descendants can acquire new genes as inserts through horizontal gene transfer.

## Module_03_Problem_set_04： “Fine-scale phylogenetic architecture”

#### Learning objectives: 

* Gain experience estimating diversity within a hypothetical microbial community

#### Outline:

In class Day 1:

1. Define and describe species within your group's "microbial" community.  
2. Count and record individuals within your defined species groups.  
3. Remix all species together to reform the original community.  
4. Each person in your group takes a random sample of the community (*i.e.* devide up the candy).

Assignment:

5. Individually, complete a collection curve for your sample.  
6. Calculate alpha-diversity based on your original total community and your individual sample.  

In class Day 2:

7. Compare diversity between groups.

#### Part 1: Description and enumeration

For your community:

* ***Construct a table listing each species, its distinguishing characteristics, the name you have given it, and the number of occurrences of the species in the collection.***

```{r}
#To make tables
library(kableExtra)
library(knitr)
#To manipulate and plot data
library(tidyverse)
# Create community data
Community = data.frame(
  number = c(1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32),
  name = c("mmgreen", "mmred", "mmblue","mmyellow","mmbrown","mmorange","sktbrown","sktred","sktgreen","sktorange","sktyellow","gmbrred","gmbrpink","gmbrgreen","gmbrorange","gmbryellow","gmbrwhite","mipink","migreen","miyellow","miorange","mired","wmred","blyellow","blgreen","blpurple","blorange","blred","chcltkiss","lgpink","lgyellow","lgblue"),
  characteristics = c("m&m green", "m&m red", "m&m blue","m&m yellow","m&m brown","m&m orange","skittle brown","skittle red","skittle green","skittle orange","skittle yellow","gummi bear red","gummi bear pink","gummi bear green","gummi bear orange","gummi bear yellow","gummi bear white","m&i pink","m&i green","m&i yellow","m&i orange","m&i red","worms red","balls yellow","balls green", "balls purple","balls orange","balls red","chacolate kiss","lego pink","lego yellow","lego blue"),
  occurences = c(28, 28, 60,44,30,63,39,33,42,35,23,15,16,18,15,19,16,39,36,27,32,40,14,4,5,3,5,7,16,7,5,4)
)

# Construct a table list for community
Community %>% 
  kable("html") %>%
  kable_styling(bootstrap_options = "striped", font_size = 10, full_width = F)
```
	
For your community:

* ***Ask yourself if your collection of microbial cells from seawater represents the actual diversity of microorganisms inhabiting waters along the Line-P transect. Were the majority of different species sampled or were many missed?***

My sample can represents the actual diversity in the community. Although four species are missed, they are not the majority species.

#### Part 2: Collector's curve

For your sample:

* ***Create a collector's curve for your sample (not the entire original community).***

```{r}
# Import data for collector's curve
Collector_curve = read.table(file="Collector_curve_data.txt", header=TRUE,sep="\t")

# Create a collector's curve
ggplot(Collector_curve, aes(x=individuals, y=species)) +
  geom_point() +
  geom_smooth() +
  labs(x="Cumulative number of individuals classified", y="Cumulative number of species observed")
```

* ***Does the curve flatten out? If so, after how many individual cells have been collected?***

The curve becomes flat after about 75 individuals have been classified.

* ***What can you conclude from the shape of your collector's curve as to your depth of sampling?***

As more and more individuals classified, the collector's curve becomes flat gradually, and eventually parallel to x axis. 

#### Part 3: Diversity estimates (alpha diversity)
Using the table from Part 1, calculate species diversity using the following indices or metrics.

##### Diversity: Simpson Reciprocal Index

* ***What is the Simpson Reciprocal Index for your sample?***

```{r}
# Import sample data
Sample= read.table(file="Sample_data.txt",header=TRUE,sep="\t")

# Calculate the Simpson Reciprocal Index (SRI) for the sample
library(dplyr)
1/sum(Sample%>%
  mutate(proportion=occurences/sum(occurences))%>%
  transmute(proportion_2=proportion^2))
```

* ***What is the Simpson Reciprocal Index for your original total community?***

```{r}
# Calculate the Simpson Reciprocal Index (SRI) for the community
1/sum(Community%>%
  mutate(proportion=occurences/sum(occurences))%>%
  transmute(proportion_2=proportion^2))
```

##### Richness: Chao1 richness estimator

* ***What is the chao1 estimate for your sample?***

```{r}
obs=Sample%>% filter(occurences!=0)%>% count()
a=Sample%>% filter(occurences==1)%>% count()
b=Sample%>% filter(occurences>=2)%>% count()
obs+a^2/(2*b) 
```

* ***What is the chao1 estimate for your original total community?***

```{r}
obs=Community%>% filter(occurences!=0)%>% count()
a=Community%>% filter(occurences==1)%>% count()
b=Community%>% filter(occurences>=2)%>% count()
obs+a^2/(2*b) 
```

#### Part 4: Alpha-diversity functions in R

For your sample:

* ***What are the Simpson Reciprocal Indices for your sample and community using the R function?***

**Sample**
```{r}
library(vegan)
# Construct a sample species table 
Sample_diversity = 
  Sample %>% 
  select(name, occurences) %>% 
  spread(name, occurences)

Sample_diversity

# Calculate Simposon Reciprocal Indices for the sample
diversity(Sample_diversity, index="invsimpson")
```

**Community**
```{r}
# Construct the community species table
Community_diversity = 
  Community %>% 
  select(name, occurences) %>% 
  spread(name, occurences)

Community_diversity

# Calculate Simpson Reciprocal Indices for the community
diversity(Community_diversity, index="invsimpson")
```

* ***What are the chao1 estimates for your sample and community using the R function?***
    
**Sample**
```{r}
# Calculate chao1 estimates for sample
specpool(Sample_diversity)
```

**Community**
```{r}
# Calculate chao1 estimates for community
specpool(Community_diversity)
```

   + Verify that these values match your previous calculations.
   
Chao1 richness estimators determined by alpha-diversity functions are the same as the values calculated in part 3.

#### Part 5: Concluding activity
If you are stuck on some of these final questions, reading the [Kunin et al. 2010](https://www.ncbi.nlm.nih.gov/pubmed/19725865) and [Lundin et al. 2012](https://www.ncbi.nlm.nih.gov/pubmed/23760801) papers may provide helpful insights. 

* ***How does the measure of diversity depend on the definition of species in your samples?***

The definition of species has significant effects of the calculated values of diversity. Species in the sample are defined based on different colors, shapes, and tastes.  However measured diversity in the sample will change if species richness and evenness are altered when different methods are used to identify species. 

* ***Can you think of alternative ways to cluster or bin your data that might change the observed number of species?***

If data is clustered based on colors, shapes or materials, observed species number will change.

* ***How might different sequencing technologies influence observed diversity in a sample?*** 

Different sequencing methods and sequencing depth may bring errors in the measurement of sample diversity. Sequencing methods with distinct stringencies on read quality filtering and identity thresholds can lead to false phylotypes detected and the recruitment of erroneous OTUs and singletons into larger clusters, resulting in artificial inflation of diversity estimates at different levels [1]. Additionally, detection on different regions (e.g. 5’-forward and 3’-reverse) can also cause errors at different levels. Kunin et al. 2010 shows that the 5’-forward region had an average of 15% more reads with one or more errors than did the 3’-reverse region when reads were treated in the same quality filtering level [1]. On the other hand, the representative of sample diversity on the community is significantly affected by sequencing depth. Lundin et al. 2012 reports that for a community with total sequences of 15000-20000, 1000 denoised sequences for each sample are sufficient to estimate trends in β-diversity accurately and precisely, while 5000 denoised sequences may be needed for α-diversity estimation [2].

*References:*  
[1] V. Kunin, A. Engelbrektson, H. Ochman and P. Hugenholtz, "Wrinkles in the rare biosphere: pyrosequencing errors can lead to artificial inflation of diversity estimates," *Environmental Microbiology*, 12(1), pp. 118-123, 2010.  
[2] D. Lundin, I. Severin, J. Brendan Logue, ?. ?stman, A. F. Andersson and E. S. Lindstr?m, "Which sequencing depth is sufficient to describe patterns in bacterial a- and b-diversity?" *Environmental Microbiology Reports*, 4 (3): 367-372, Mar. 2012.  

## Module_03_Writing_Assignment_03

#### Prompt

*“Discuss the challenges involved in defining a microbial species and how HGT (Horizontal Gene Transfer) complicates matters, especially in the context of the evolution and phylogenetic distribution of microbial metabolic pathways. Can you comment on how HGT influences the maintenance of global biogeochemical cycles through time? Finally, do you think it is necessary to have a clear definition of a microbial species?  Why or why not?”*

####Introducation 

Since the discovery of microorganisms by Robert Hooke and Antoni van Leeuwenhoek in the 17th century, we have known a lot on microbiology and obtained numerous benefits from this discipline by applying it to various industrial fields like medical industry, biotechnological industry, and environmental engineering [1]. However there are still several fundamental questions controversial in microbiology that have not had clear answers widely accepted so far. One of them is how to define a microbial species. Until now, we do not have a specific set of criteria on defining microbial species, which leads to different and incomparable results in microbiological studies, and thus obstructs our deep-in understanding on microbial world as well as application of these studies. Therefore, in this essay, the challenges involved in defining microbial species will be discussed. After that, the effects of horizontal gene transfer (HGT) on this issue will be investigated as well as the role of HGT playing in the maintenance of global biogeochemical cycles through time. Finally, the significance of a clear definition of a microbial species will be estimated to demonstrate the necessity to resolve this problem.   

####The challenges involved in defining a microbial species

Defining microbial species is not only an important academic exercise, but also has major practical applications in many fields, such as infectious disease diagnoses and microbial contaminants determination [2]. However, microbial species identification is a particularly challenging issue for us today, even though we have already acquired technology like DNA sequencing for microbiological studies at molecular level. The difficulty is mainly caused by the absent of official definition on microbial species concept, the continuum of bacterial genome sequence diversity, and high frequency of horizontal gene transfer among microorganisms.   

Due to the lack of agreement on microbial species concept, a single microorganism can be identified as different species using different sets of criteria [3]. Although the official definition of microbial species concept is absent, there are several distinct methods proposed for microbial species identification. For example, taxospecies refers to a group of organisms with mutually high phenotypic similarity that form an independent phenotypic cluster, while genospecies is defined as a group of organisms presenting high DNA-DNA hybridization values [3, 4]. However, these species identification methods do not integrate both genomic and phenotypic characteristics [4]. As a result, microorganisms with distinct genome sequences but showing the same phenotypes would be defined as the same taxospecies, whereas organisms with different phenotypes but possessing high DNA-DNA hybridization would be regarded as a genospecies. Therefore, a group of microorganisms can be defined as different species based on different identification approaches. 

Another challenge on defining microbial species is the continuum of microbial genome sequence diversity [2]. Although nowadays scientific technology has such dramatic development that we can apply it in microbiology to sequence and compare thousands of complete microbial genomes, boundaries among microbial species remain unclear because microbial genome diversity seems not to be discrete like eukaryotic genome sequences [2]. For instance, human and chimpanzee share a surprisingly high DNA similarity as 98.8%, but they are defined as two different species since their genomic sequences distribute into two discrete and well-separated populations [2, 5]. However, microbial genomes has been observed to fall between *B. anthracis* and *B. cereus* which only have 2-3% difference in average nucleotide identity [2]. Consequently, it is hard to define these bacteria as *B. anthracis*, or *B. cereus*, or other species.

Additionally, horizontal gene transfer (HGT) is a large obstacle on defining microbial species. HGT, defined as the non-genealogical transmission of genetic material from one organism to another, has been considered as one of major sources for microorganisms acquiring new genes and functions [6, 7]. Hence, HGT increases the hardness to construct a correct phylogenetic tree as it confounds the phylogenetic signals in metabolic and molecular markers [7]. Besides, even if a phylogenetic tree is constructed successfully, it may become unsustainable because very few bacterial genes (less than 0.7%) are reported as core genes [7, 8]. The impacts of HGT on defining microbial species will be discussed in details in the following part.  

####How horizontal gene transfer (HGT) complicates the issue of defining a microbial species?

Currently, the reconstruction of phylogenetic relationships based on molecular sequence data has been regarded as the most reliable way to identify microbial species [3]. However, HGT brings challenges to this method because it allows different species to possess the same gene sequences, which confounds phylogenetic trees. On the other hand, it is difficult to predict horizontally transferred genes in a complete set of genomic, since both informational and operational genes can be transferred horizontally and HGT can occur between as well as among the two microbial domains: Bacteria and Archaea.  

HGT potentially confounds phylogenetic signals leading to incorrect inference of phylogenetic trees, when phylogenetic trees are constructed based on one gene sequence. Two distantly related bacteria having a gene exchanged may be difficultly differentiated by the phylogenetic tree which shows that the two bacteria are closely related. Therefore, it would be more reliable to use a wide range of gene sequences for phylogenetic analysis. On the other hand, it has been proposed that 16S rRNA gene, an effective tool for phylogenetic trees construction, can be transferred horizontally [7, 9]. Badger et al. finds that 16S rRNA gene sequence phylogeny defines *Hyphomonas neptunium* as a member of ‘*Rhodobacterales*’, completely different from the result of other gene sequence methods as ‘*Caulobacterales*’, which allows researchers to put forward a potential explanation that 16S rRNA might be transferred horizontally [9]. If this explanation was proved to be correct, the validity of phylogenetic trees constructed by 16S rRNA should be reevaluated [10].

On the other hand, the distinct frequency of HGT on different functional genes and in various directions makes it hard to predict horizontal transferred genes. Several studies indicate that informational genes have less likelihood to HGT than operational genes, but the difference is small, and the participation of operational genes in HGT is not equal [7]. Moreover, the evolutionary distance between organisms is not a critical factor of HGT. Some studies reflect that HGT occur more frequently for short and intermediate distances, but HGT may take place between distantly related organisms [7]. Others show that the major direction of HGT in prokaryotic world is from Bacteria to Archaea, while there are some gene transfer events from Archaea to Bacteria [7]. Therefore, the high prevalence of HGT among microorganisms enhances the difficulty of microbial species identification through phylogenetic trees.              
 
####The effects of HGT on the maintenance of global biogeochemical cycles through time?

Although HGT complicates the issue of defining microbial species, it plays a significant role in the maintenance of global biogeochemical cycles through evolutionary time. HGT is one of the major factors that keep core genes, coding the major redox reactions important for biogeochemical cycles, relatively stable over time [11]. On the other hand, new genes acquired through HGT may enable microorganisms to adapt and survive in new environment, and thus core genes in the microbial genome are able to be preserved and spread continuously to other microorganisms via horizontal transfer [12]. 

The widely global distribution of core genes responsible for essential metabolic processes among microorganisms demonstrates the substantially positive effects of HGT on the maintenance of global biogeochemical cycles. There are abundant examples, and one of them is nitrogenases gene. It has been reported that nitrogenases gene was probably transferred from an Archaeal species to oxygenic photosynthetic cyanobacteria first, and then widely spread among Bacteria and Archaea [11]. This wide distribution helps core genes survive massive extinction period, since as long as one microbe containing these core genes survived, these core genes could be transferred again to other microbes horizontally and vertically when environment became favorable. Furthermore, when a certain microbial species become extinct in a specific ecosystem, other microbial species can replace its position in biogeochemical cycles as they carry the same core genes and perform the same functions as the extinct one.

Besides, HGT of non-core genes is another method to maintain global biogeochemical cycles. Although the acquisition of non-core genes does not confer new essential functions for biogeochemical cycles to recipients, it may allow recipients to evolve or survive in new environments so that core genes carried by recipients can be preserved. For instance, bacterium *Neisseria gonorrhoeae* obtained a new gene from a human cell which enables the organism to evolve and adapt to human body, as a result, the core gene for the catabolic process on organic carbon in the *Neisseria gonorrhoeae* is preserved and continuously contributes to the global carbon cycle [12].                  

####The necessities of a clear definition of a microbial species

For the question that whether it is necessary to have a clear definition of microbial species, my answer is yes. The first reason is that a clear definition on microbial species concept can deepen our understanding on microbiology. Using the same criteria for defining microbial species enables microbiologists to obtain more reliable, comparable, and reproducible results in their research. Additionally, without a clear definition of microbial species, it would be more difficult to educate the public about bacteria, such as which bacterium is beneficial or detrimental to human health. On the other hand, the clear definition of microbial species brings far more conveniences in the practical applications of microbiology. For example, in order to treat different diseases correctly caused by distinct bacteria like *Bacillus anthracis* and *Bacillus cereus*, we need to name the two species to connect them with their corresponding symptoms [2]. Another typical example is environmental monitoring. Microbial parameters are essential for water quality assessment. If we do not have a widely accepted definition of microbial species, the concentration of harmful bacteria like *E. coli* would be hard to determine in water samples, and thus to evaluate water quality objectively. Therefore, a clear definition on microbial species is highly necessary in order to enhance our comprehension on microbial world and future application of microbiology.   

####Conclusions

A clear definition on microbial species is very useful not only for microbial studies but also for the practical application of microbiology. However, defining microbial species is a challenging issue because of the absent of official definition on microbial species concept, the continuum of bacterial genome sequence diversity, and high frequency of horizontal gene transfer (HGT) in microbial world. Among these reasons, HGT complicates this issue a lot, since its potential to confound phylogenetic signals for the construction of phylogenetic trees and difficulties to predict horizontally transferred genes due to different frequency of HGT in different situations. However, HGT has huge contributions to the maintenance of global biogeochemical cycles, as it keeps core genes for biogeochemical cycles relatively stable and enables microbes carrying core genes to adapt and survive in new environments.    

####References

[1] Gest, H. 2004. The discovery of microorganisms by Robert Hooke and Antoni Van Leeuwenhoek, fellows of the royal society. *Notes Rec. R. Soc. Lond.*, 58 (2), pp.187–201.  
[2] Konstantinidis, K., Kolter, R. 2017. Do bacterial species really exist and why should we care? [Online]. Link: http://schaechter.asmblog.org/schaechter/2017/01/do-bacterial-species-really-exist-and-why-should-we-care.html.   
[3] O. Ogunseitan (Ed.). “The Concept of Microbial Species,” in *Microbial Diversity: Form and Function in Prokaryotes*. Malden, MA: Blackwell Science Ltd, 2004, pp. 3-21.    
[4] Rossello-Mora, R., Amann, R. 2001. The species concept for prokaryotes. *FEMS Microbiology Reviews*, 25, pp. 39-67.   
[5] American Museum of Natural History. DNA: Comparing Humans and Chimps [Online]. Link: https://www.amnh.org/exhibitions/permanent-exhibitions/human-origins-and-cultural-halls/anne-and-bernard-spitzer-hall-of-human-origins/understanding-our-past/dna-comparing-humans-and-chimps/.   
[6] Goldenfeld, N. & Woese, C. 2007. Biology’s next revolution. *Nature*, 445, pp.369.  
[7] Boto, L. 2009. Horizontal gene transfer in evolution: facts and challenges. *Proc. R. Soc. B*, 277, pp. 819–827.  
[8] Bapteste, E. & Boucher, Y. 2008 Lateral gene transfer challenges principles of microbial systematic. *Trends Microbiol*. 16, pp. 200–207.  
[9] Badger, J. H., Eisen, J. A., Ward, N. L. 2005. Genomic analysis of Hyphomonas neptunium contradicts 16S rRNA gene-based phylogenetic analysis: implications for the taxonomy of the orders ‘*Rhodobacterales*’ and *Caulobacterales*. *International Journal of Systematic and Evolutionary Microbiology*, 55, pp. 1021–1026.  
[10] Horizontal gene transfer. Link: https://en.wikipedia.org/wiki/Horizontal_gene_transfer#cite_note-Lawton-97.  
[11] Falkowski, P. G., Fenchel, T., Delong, E. F. 2008. The microbial engines that drive earth’s biogeochemical cycles. *Science*, 320, pp. 1034-1039.   
[12] Rogers, K. Horizontal gene transfer [Online]. Link: https://www.britannica.com/science/horizontal-gene-transfer.   

## Module_03_Additional_Readings

[1] S. J. Giovannoni, "Vitamins in the sea", *PNAS*, vol. 109, no. 35, pp. 13888-13889, Aug. 2012. (http://www.pnas.org/content/109/35/13888)

[2] M. Torres-Beltr?n, A. K. Hawley, D. Capelle, E. Zaikova, D. A. Walsh, A. Mueller, M. Scofield, C. Payne, L. Pakhomova, S. Kheirandish, J. Finke, M. Bhatia, O. Shevchuk, E. A. Gies, D. Fairley, C. Michiels, C. A. Suttle, F. Whitney, S. A. Crowe, P. D. Torte, and S. J. Hallam, "A compendium," *Nature*, pp. 1-10, Oct. 2017. (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5663218/)

[3] A. K. Hawley, M. Torres-Beltr?n, E. Zaikova, D. A. Walsh, A.  Mueller, M. Scofield, S. Kheirandish, C. Payne, L. Pakhomova, M. Bhatia, O. Shevchuk, E. A. Gies, D. Fairley, S. A. Malfatti, A. D. Norbeck, H. M. Brewer, L. Pasa-Tolic, T. G. Rio, C. A. Suttle, S. Tringe, and S. J. Hallam, "A compendium of multi-omic sequence information from the Saanich Inlet water column," *Nature*, pp. 1-13, Oct. 2017. (https://www.nature.com/articles/sdata2017160)

[4] R. A. Welch, V. Burland, G. Plunkett, P. Redford, P. Roesch, D. Rasko, E. L. Buckles, S.-R. Liou, A. Boutin, J. Hackett, D. Stroud, G. F. Mayhew, D. J. Rose, S. Zhou, D. C. Schwartz, N. T. Perna, H. L. T. Mobley, M. S. Donnenberg, and F. R. Blattner, "Extensive mosaic structure revealed by the complete genome sequence of uropathogenic Escherichia coli," *PNAS*, vol. 99, no. 26, pp. 17020-17024, Dec. 2002. (http://www.pnas.org/content/99/26/17020.long)

[5] B. J. Callahan, P. J McMurdie and S. P Holmes, "Exact sequence variants should replace operational taxonomic units in marker-gene data analysis," *nature*, pp. 2639-2643, July 2017.(http://www.nature.com/articles/ismej2017119)

[6] A. D. Gaudet., L. M. Ramer, J. Nakonechny, J. J. Cragg, M. S.
Ramer, "Small-Group Learning in an Upper-Level University Biology Class Enhances Academic Performance and Student Attitudes Toward Group Work," *PLOS ONE*, vol. 5, issue 12, pp. 1-10, Dec. 2010. (http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0015821)

[7] D. Lundin, I. Severin, J. Brendan Logue, ?. ?stman, A. F. Andersson and E. S. Lindstr?m, "Which sequencing depth is sufficient to describe patterns in bacterial a- and b-diversity?" *Environmental Microbiology Reports*, 4 (3): 367-372, Mar. 2012. (https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1758-2229.2012.00345.x)

[8] M. L. Sogin, H. G. Morrison, J. A. Huber, D. M. Welch, S. M. Huse, P. R. Neal, J. M. Arrieta, and G. J. Herndl, "Microbial diversity in the deep sea and the underexplored 'rare biosphere'," *PNAS*, vol. 103, no. 32, pp. 12115-12120, Aug. 2006. (http://www.pnas.org/content/103/32/12115.long)

[9] O. X. Corderoa, L. Ventourasb, E. F. DeLonga, and M. F. Polza, "Public good dynamics drive evolution of iron acquisition strategies in natural bacterioplankton populations," *PNAS*, Aug. 2012. (http://www.pnas.org/content/109/49/20059/tab-figures-data)

[10] S. J. Hallam, M. Torres-Beltr?n, and A. K. Hawley, "Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone," *Nature*, Oct. 2017. (https://www.nature.com/articles/sdata2017158)

[11] V. Kunin, A. Engelbrektson, H. Ochman and P. Hugenholtz, "Wrinkles in the rare biosphere: pyrosequencing errors can lead to artificial inflation of diversity estimates," *Environmental Microbiology*, 12(1), pp. 118-123, 2010. (https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1462-2920.2009.02051.x)

[12] J. R. Thompson, S. Pacocha, C. Pharino, V. Klepac-Ceraj, D. E. Hunt, J. Benoit, "Genotypic Diversity Within a Natural Coastal Bacterioplankton Population," *Science*, vol. 307, pp. 1311-1312, Feb. 2005. (http://science.sciencemag.org/content/307/5713/1311.long)

[13] J. J. Morris, R. E. Lenski, and E. R. Zinserc, "The Black Queen Hypothesis: Evolution of Dependencies through Adaptive Gene Loss," *mBio*, vol. 3, issue 2, pp. 1-7, Apr. 2012. (http://mbio.asm.org/content/3/2/e00036-12.abstract)

## Project_01

### Project_01_Report

#### Authors

Team 1: Albert Chang (), Alison Fong (33399149), Karen Lau (16524143), Yaqian
 Luo (59751503), Jessica Ngo (31837131), Peter (Kiet) Truong (36645133)

####Abstract  

To study the diversity and biochemical responses of microbial communities in the context of oxygen minimum zones (OMZs), Saanich Inlet was used as a model ecosystem from which water samples were collected at seven major depths spanning the oxycline. A metagenomic study was conducted in which genomic DNA was extracted from the water samples, PCR amplified, assembled into contiguous sequences, and processed into operational taxonomic units (OTUs) and amplicon sequence variants (ASVs) using mothur and QIIME2 pipelines. Based on OTU and ASV results, we chose to focus in on Cyanobacteria as our taxon of interest. Both mothur and QIIME2 data produced alpha diversity values based on Shannon's diversity index that suggest a decrease of Cyanobacterial abundance as depth increases. In addition, Cyanobacterial abundance significantly differs across depth and oxygen concentration according to both mothur and QIIME2 data; specifically, abundance significantly decreases at deeper depths and in environments with lower concentrations of oxygen. Within the Cyanobacteria phylum, there are 15 OTUs and 51 ASVs across all samples. Abundance of five OTUs from the mothur pipeline showed significant changes across both depth and oxygen. In contrast, abundance of none of the ASVs from the QIIME2 pipeline showed significant changes across depth, although 17 ASVs showed significant changes across oxygen concentrations. An increase of Cyanobacterial abundance at shallow depths may be explained in part by their ability to absorb red and orange light at the upper boundaries of the water column. This is also supported by the significant change in oxygen and chlorophyll A concentrations across the depth profile. Another explanation for low Cyanobacterial abundance at lower depths may be due to changes in temperature; where temperature drops below 10^o^C at 100m, Cyanobacterial growth stops completely.To study the diversity and biochemical responses of microbial communities in the context of oxygen minimum zones (OMZs), Saanich Inlet was used as a model ecosystem from which water samples were collected at seven major depths spanning the oxycline. A metagenomic study was conducted in which genomic DNA was extracted from the water samples, PCR amplified, assembled into contiguous sequences, and processed into operational taxonomic units (OTUs) and amplicon sequence variants (ASVs) using mothur and QIIME2 pipelines. Based on OTU and ASV results, we chose to focus in on Cyanobacteria as our taxon of interest. Both mothur and QIIME2 data produced alpha diversity values based on Shannon's diversity index that suggest a decrease of Cyanobacterial abundance as depth increases. In addition, Cyanobacterial abundance significantly differs across depth and oxygen concentration according to both mothur and QIIME2 data; specifically, abundance significantly decreases at deeper depths and in environments with lower concentrations of oxygen. Within the Cyanobacteria phylum, there are 15 OTUs and 51 ASVs across all samples. Abundance of five OTUs from the mothur pipeline showed significant changes across both depth and oxygen. In contrast, abundance of none of the ASVs from the QIIME2 pipeline showed significant changes across depth, although 17 ASVs showed significant changes across oxygen concentrations. An increase of Cyanobacterial abundance at shallow depths may be explained in part by their ability to absorb red and orange light at the upper boundaries of the water column. This is also supported by the significant change in oxygen and chlorophyll A concentrations across the depth profile. Another explanation for low Cyanobacterial abundance at lower depths may be due to changes in temperature; where temperature drops below 10^o^C at 100m, Cyanobacterial growth stops completely.  

####Introduction  

Oxygen minimum zones (OMZs) are areas in the ocean where dissolved oxygen concentrations fall below 20 $\mu$M (1). Due to temperature increases and other effects caused by global warming, OMZs are expanding at a notable rate. Saanich Inlet, a seasonally anoxic fjord off the coast of British Columbia, is a model ecosystem for studying the diversity and biochemical responses of microbial communities to the hypoxic environments commonly observed in OMZs (1, 2). In particular, Saanich Inlet has been used to model the metabolic coupling and symbiotic interactions that occur in OMZs (3).  The inlet undergoes recurring cycles of water column stratification and deep water renewal, rendering it a model ecosystem for studying microbial responses to changes in ocean deoxygenation levels (4). Increased levels of primary productivity in ocean surfaces during the spring season, as well as the limited mixing which occurs between the basin and surface waters both result in the development of an anoxic body of water with increasing depth in the Inlet (2). These anoxic regions become populated with chemolithoautotrophs, and eventually lead to a decrease in aerobically respiring organisms found deeper within these zones. Past studies have demonstrated that these kinds of metabolic shifts generally lead to a loss of nitrogen along with the production of greenhouse gases, most notably methane (CH~4~) and nitrous oxide (N~2~O) (1).  

In order to investigate the changes that occur in these OMZs, water samples of various depths were collected from Saanich Inlet. Genomic DNA was extracted from these to conduct a metagenomics study, allowing to overcome the barrier of uncultivability of these samples and enable a more thorough exploration of the relationship which exists between the microbes and their communities based on genetic distribution of metabolic processes (5, 6, 7, 8). The extracted DNA is sequenced to generate raw data, which can then be assembled into contiguous sequences. These contigs generated by amplicon sequencing are then compared to a sequence database to determine the microbial taxa present in the environment at each water depth. This involves processing the sequencing data, and there currently exists two methods for this type of data analysis: operational taxonomic units (OTUs) and amplicon sequence variants (ASVs). OTU based pipelines work based on clustering reads which differ by less than a fixed dissimilarity threshold (9). This allows more data to be kept, although some may not be representative of the actual taxa in the community. On the contrary, ASV based pipelines resolve these sequence variants by inferring biological sequences in the sample prior to amplification and possible sequencing errors, and are able to distinguish variants which differ by even one nucleotide (9). This treats each ASV as individual species, though has the potential to discard more data and bias towards sequences that are less error-prone.  

The objective of this paper is to analyze the data generated from both OTU and ASV pipelines in order to decide which produces more logical inferences, and ultimately determining which pipeline would be preferred to carry out future analysis of collected water samples. The taxonomy of interest which was selected for this comparison was the phylum Cyanobacteria. Cyanobacteria was selected as there are sufficient numbers of OTUs and ASVs to make sound comparisons, but not so much so that computation-wise it would be infeasible.  

####Methods  

#####Sampling  

Samples were obtained on Saanich Inlet Cruise 72 and taken from seven major depths spanning the oxycline: 10, 100, 120, 135, 150, 165 and 200 m. Waters were filtered, and genomic DNA was extracted. Further sampling details can be found in (3).  

#####DNA Sequencing 

Samples were PCR amplified using the 515F and 808R primers, then sequenced according to the standard operating protocol on an Illumina MiSeq platform with Phred33 quality scores.  

#####Data Processing
Sequences were processed using either mothur or QIIME2 as follows:  

*Mothur Pipeline*: mothur was used to clean-up the data. In brief, paired end reads were combined into contigs using their overlapping regions. Low quality sequences, useless sequence data, chimeric sequences and singletons were removed. OTUs were then determined at 97% similarity. OTUs were classified using the SILVA databases, and the taxonomies for each OTU were condensed. The OTU table, taxonomy data and sample metadata were subsequently cleaned up and combined into a phyloseq object.  

*QIIME2 Pipeline*: Demultiplexed sequences were imported into QIIME as manifest reads. QIIME was used to clean up the data along with ASV determination in one step. Sequence quality was visually evaluated, and sequence quality trimming was conducted. All other trimming/filtering parameters were left as default. ASV determination was completed using the Dada2 protocol. ASV classification was completed using the Silva version 119 database at 99% similarity. The ASV table, taxonomy data and sample metadata were subsequently cleaned up and combined into a phyloseq object.  

#####Data Analysis   

The aforementioned phyloseq objects were imported into R version 3.4.3 (Windows) or 1.1.383 (Mac). The tidyverse, phyloseq, magrittr, knitr and cowplot packages were loaded and used to complete the data analysis. Data was piped into linear models and ANOVA tests to determine statistical significance at the 95% confidence level.  

######Environment setup and Data Cleaning

```{r, echo=FALSE, "loading packages and data"}
library("tidyverse")
library("phyloseq")
library("magrittr")
library("knitr")
library("cowplot")

load("./mothur_phyloseq.RData")
load("./qiime2_phyloseq.RData")
```

```{r, echo=FALSE}
knitr::opts_chunk$set(error = TRUE)
```

```{r, echo=FALSE, "data cleaning"}
#Mothur data
set.seed(4832)
m.norm = rarefy_even_depth(mothur, sample.size=100000)
m.perc = transform_sample_counts(m.norm, function(x) 100 * x/sum(x))

#Qiime2 data
set.seed(4832)
q.norm = rarefy_even_depth(qiime2, sample.size=100000)
q.perc = transform_sample_counts(q.norm, function(x) 100 * x/sum(x))

```

####Results  

1. How does microbial community structure change with depth and oxygen concentration?  

**Alpha-diversity comparison:**   
*Mothur*  
As shown in Figure 1, alpha-diversity by Shannon's diversity index based on mothur shows an overall decreasing trend as depth increases. Specifically, a stable trend is observed from depths of 0-100m, starts decreasing at 100-150m and stabilizes at 150-200m. Figure 1 also shows an overall increasing trend as oxygen increases. Specifically, Shannon's diversity index increases from 2.3-4.25 at oxygen concentrations of 0-40$\mu$M and decreases from 4.25-3.9 at oxygen concentrations of 40-220?M. It was also observed that Shannon's diversity index is higher in oxic conditions (3.84 ? 0.45) than anoxic conditions (2.39 ? 0.07).  

*QIIME2*  
The trends of Shannon's diversity index across depth and oxygen based on QIIME2 data are similar to those of mothur data. However, QIIME2 pipeline produces higher Shannon's diversity index than mothur does. Shannon's diversity index increases from 2.9-5.2 at oxygen concentrations of 0-40$\mu$M and decreases from 5.2-5.1 at oxygen concentrations of 40-200m. Figure 2 shows that Shannon's diversity index is higher in oxic conditions (4.80 ? 0.43) than anoxic conditions (3.15 ? 0.18).  

######Alpha-diversity of mothur data
```{r, echo=FALSE, "Question 1a_1", fig.width=14, fig.height=10}
#Mothur data
m.alpha = estimate_richness(m.norm, measures = c("Chao1", "Shannon"))
m.meta.alpha = full_join(rownames_to_column(m.alpha), rownames_to_column(data.frame(m.perc@sam_data)), by = "rowname")

#Mother data - Alpha diversity across depth
plot_1=m.meta.alpha %>%
ggplot() +
   geom_point(aes(x=Depth_m, y=Shannon)) +
   geom_smooth(method='auto', aes(x=as.numeric(Depth_m), y=Shannon)) +
   labs(y="Shannon's diversity index", x="Depth (m)")

#Mothur data - Alpha diversity across oxygen
plot_2=m.meta.alpha %>%
ggplot() +
  geom_point(aes(x=O2_uM, y=Shannon)) +
  labs(y="Shannon's diversity index", x="Oxygen (uM)")

#Mothur data - Alpha diversity by oxic/anoxic
plot_3=m.meta.alpha %>%
  mutate(O2_group = ifelse(O2_uM == 0, "anoxic", "oxic")) %>%
ggplot() +
  geom_boxplot(aes(x=O2_group, y=Shannon)) +
  labs(y="Shannon's diversity index", x="Oxygen")

plot_grid(plot_1, plot_2, plot_3, labels=c("A", "B","c"))+
   labs(caption="Figure 1. Alpha-diversity across depth or oxygen using mothur data. A) Alpha-diversity across depth using mothur data.
        B) Alpha-diversity across oxygen using mothur data. C) Alpha-diversity by oxic/anoxic using mothur data")+
  theme(plot.caption = element_text(size=rel(1.2)))

# Average and standard deviation of mothur by oxic/anoxic
m.oxygen=m.meta.alpha %>%
  mutate(O2_group = ifelse(O2_uM == 0, "anoxic", "oxic")) 

m.anoxic=m.oxygen%>%
  filter(O2_group=="anoxic")
  
ave.m.anoxic=mean(m.anoxic$Shannon)
st.m.anoxic=sd(m.anoxic$Shannon)

m.oxic=m.oxygen%>%
  filter(O2_group=="oxic")

ave.m.oxic=mean(m.oxic$Shannon)
st.m.oxic=sd(m.oxic$Shannon)

m.avg_std = data.frame(
  Statistic=c("Average","Standard deviation"),
  Oxic=c(ave.m.oxic, st.m.oxic),
  Anoxic=c(ave.m.anoxic, st.m.anoxic)
)

kable(m.avg_std,captio="Table 1. Average and standard deviation of alpha-diversity by oxic/anoxic with mothur data", align = "c")

```

######Alpha-diversity of QIIME2 data
``` {r, echo=FALSE, "Question 1a_2", fig.width=14, fig.height=10}
#Qiime2 data
q.alpha = estimate_richness(q.norm, measures = c("Chao1", "Shannon"))
q.meta.alpha = full_join(rownames_to_column(q.alpha), rownames_to_column(data.frame(q.perc@sam_data)), by = "rowname")

#Qiime 2 data - Alpha diversity across depth
plot_4=q.meta.alpha %>%
 
ggplot() +
   geom_point(aes(x=Depth_m, y=Shannon)) +
   geom_smooth(method='auto', aes(x=as.numeric(Depth_m), y=Shannon)) +
   labs(y="Shannon's diversity index", x="Depth (m)")
 
#Qiime2 data - Alpha-diversity across oxygen
plot_5=q.meta.alpha %>%
ggplot() +
  geom_point(aes(x=O2_uM, y=Shannon)) +
  labs(y="Shannon's diversity index", x="Oxygen (uM)") 
 
#Qiime2 data - Alpha-diversity by oxic/anoxic
plot_6=q.meta.alpha %>%
  mutate(O2_group = ifelse(O2_uM == 0, "anoxic", "oxic")) %>%
ggplot() +
  geom_boxplot(aes(x=O2_group, y=Shannon)) +
  labs(y="Shannon's diversity index", x="Oxygen")
  
plot_grid(plot_4, plot_5, plot_6, labels=c("A", "B","c"))+
   labs(caption="Figure 2. Alpha-diversity across depth or oxygen using QIIME2 data. A) Alpha-diversity across depth using QIIME2 data. 
        B) Alpha-diversity across oxygen using QIIME2 data. C) Alpha-diversity by oxic/anoxic using QIIME2 data")+
  theme(plot.caption = element_text(size=rel(1.2)))

# Average and standard deviation of quiime2 by oxic/anoxic
q.oxygen=q.meta.alpha %>%
  mutate(O2_group = ifelse(O2_uM == 0, "anoxic", "oxic")) 

q.anoxic=q.oxygen%>%
  filter(O2_group=="anoxic")

ave.q.anoxic=mean(q.anoxic$Shannon)
sd.q.anoxic=sd(q.anoxic$Shannon)

q.oxic=q.oxygen%>%
  filter(O2_group=="oxic")

ave.q.oxic=mean(q.oxic$Shannon)
sd.q.oxic=sd(q.oxic$Shannon)

q.avg_std = data.frame(
  Statistic=c("Average","Standard deviation"),
  Oxic=c(ave.q.oxic, sd.q.oxic),
  Anoxic=c(ave.q.anoxic, sd.q.anoxic)
)

kable(q.avg_std,captio="Table 2. Average and standard deviation of alpha-diversity by oxic/anoxic with QIIME2 data", align = "c")

```

**Taxa presence and abundance:**  

*Mothur* 
31 taxa in the phylum level are detected by mothur pipeline (Figure 3). These taxa, however, have abundance at different magnitudes. Among of them, Proteobacteria is the most predominant phylum in all samples with the highest average abundance over 75. On the contrary, phylum Peregrinlbacteria has abundance no larger than 0.001 in the seven samples (Figure 3). Additionally, different taxa have distinct changes in abundance across depth. For instance, both Thaumarchaeota and Verrucomicrobia reach their maximum abundance at depth of 100m and and gradually decrease in abundance until 200m, while Latescibacteria and Fibrobacteres are almost undetectable in shallow water and their abundances increase dramatically at depth of 200m.  

*QIIME2*   
QIIME2 pipeline detects 29 known taxa and unknown taxa in phylum level (Figure 4).  Proteobacteria is still the most abundant phylum across samples. Taxa Chlorobi and Candidate division OP3 have the smallest abundance no larger than 0.004. Different pipelines may result in different changes in abundance for the common taxa shared by mothur and QIIME2 data. Although the change patterns of phylum Actinobacteria in abundance across depth are the same in both datasets, Chloroflexi abundance increases gradually with depth in QIIME2 data different from its change pattern in mothur data, in which abundance declines at depth from 100m to 135m and increases gradually until 200m.  

```{r, echo=FALSE, "Question 1b_1", fig.width=19, fig.height=15}
#Mothur data - Phyla across samples
m.perc %>%
plot_bar() +
  geom_bar(aes(fill=Phylum), stat="identity") +
  facet_wrap(~Phylum, scales="free_y")+
  theme(legend.position="none")+
  labs(caption="Figure 3. Taxa within phylum level across samples using mothur data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.7)))
```

```{r, echo=FALSE, "Question 1b_2", fig.width=18, fig.height=13}
#Qiime2 data - Phyla across samples
q.perc %>%
plot_bar() +
  geom_bar(aes(fill=Phylum), stat="identity") +
  facet_wrap(~Phylum, scales="free_y")+
  theme(legend.position="none")+
  labs(caption="Figure 4. Taxa within phylum level across samples using qiime2 data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.7)))
```


2. Does your taxon of interest significantly differ in abundance with depth and/or oxygen concentration?  

*Mothur*   
The difference in cyanobacteria abundance within depth or oxygen was estimated by the linear model using mothur processed data. The statistical results show that abundance of cyanobacteria is significantly different with depth (p = 0.01263) and oxygen (p = 0.00012). However linear models in Figure 5 indicate completely distinct trends of cyanobacteria abundance across depth and oxygen, where there is a decrease in abundance as depth increases and an increase in abundance as oxygen concentrations increases respectively.  

*QIIME2*   
For data processed by QIIME2 pipeline, ANOVA tests indicate cyanobacteria abundance in the seven samples has significantly difference across depth (p = 0.014) and oxygen (p = 0.013).The linear models in Figure 6 reveal that cyanobacteria abundance decreases at deeper water or in the environment with insufficient oxygen.  

```{r, echo=FALSE, "Question 2a", fig.width=10, fig.height=4}
#Calculating significance with linear model for depth
m.norm %>% 
  subset_taxa(Phylum=="Cyanobacteria") %>% 
  tax_glom(taxrank = 'Phylum') %>%
  psmelt() %>%
  
  lm(Abundance ~ Depth_m, .) %>% 
  summary()

#Plot to go along with linear model above
plot_7=m.perc %>% 
  subset_taxa(Phylum=="Cyanobacteria") %>% 
  psmelt() %>% 
  group_by(Sample) %>% 
  summarize(Abundance_sum=sum(Abundance), Depth_m=mean(Depth_m)) %>% 
  
  ggplot() +
  geom_point(aes(x=Depth_m, y=Abundance_sum)) +
  geom_smooth(method='lm', aes(x=as.numeric(Depth_m), y=Abundance_sum)) 

#Calculating significance with linear model for oxygen
m.norm %>% 
  subset_taxa(Phylum=="Cyanobacteria") %>% 
  tax_glom(taxrank = 'Phylum') %>% 
  psmelt() %>% 
  
  lm(Abundance ~ O2_uM, .) %>% 
  summary()

#Plot to go along with linear model above
plot_8=m.perc %>% 
  subset_taxa(Phylum=="Cyanobacteria") %>% 
  psmelt() %>% 
  group_by(Sample) %>% 
  summarize(Abundance_sum=sum(Abundance), O2_uM=mean(O2_uM)) %>% 
  
  ggplot() +
  geom_point(aes(x=O2_uM, y=Abundance_sum)) +
  geom_smooth(method='lm', aes(x=as.numeric(O2_uM), y=Abundance_sum)) 

plot_grid(plot_7, plot_8, labels=c("A", "B"))+
  labs(caption="Figure 5. Abundance of Cyanobacteria across depth/oxygen concentrations using mothur OTUs. 
       A) Abundance of Cyanobacteria across depth. B) Abundance of Cyanobacteria across oxygen concentrations.")+
  theme(plot.caption = element_text(hjust=0.5,size=rel(0.8)))
```


```{r, echo=FALSE, "Question 2b", fig.width=10, fig.height=4}

#Calculating significance with linear model for depth
q.norm %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  tax_glom(taxrank = 'Phylum') %>%
  psmelt() %>%
  
  lm(Abundance ~ Depth_m, .) %>%
  summary()

#Plot to go along with linear model above
plot_9=q.perc %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  psmelt() %>%
  group_by(Sample) %>%
  summarize(Abundance_sum=sum(Abundance), Depth_m=mean(Depth_m)) %>%
 
ggplot() +
  geom_point(aes(x=Depth_m, y=Abundance_sum)) +
  geom_smooth(method='lm', aes(x=as.numeric(Depth_m), y=Abundance_sum)) 

#Calculating significance with linear model for oxygen
q.norm %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  tax_glom(taxrank = 'Phylum') %>%
  psmelt() %>%
  lm(Abundance ~ O2_uM, .) %>%
  summary()

#Plot to go along with linear model above
plot_10=q.perc %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  psmelt() %>%
  group_by(Sample) %>%
  summarize(Abundance_sum=sum(Abundance), O2_uM=mean(O2_uM)) %>%
 
ggplot() +
  geom_point(aes(x=O2_uM, y=Abundance_sum)) +
  geom_smooth(method='lm', aes(x=as.numeric(O2_uM), y=Abundance_sum)) 

plot_grid(plot_9, plot_10, labels=c("A", "B"))+
  labs(caption="Figure 6. Abundance of Cyanobacteria across depth/oxygen concentrations using QIIME2 ASVs. 
       A) Abundance of Cyanobacteria across depth. B) Abundance of Cyanobacteria across oxygen concentrations.")+
  theme(plot.caption = element_text(hjust=0.5,size=rel(0.8)))
```

3. Within your taxon, what is the richness (number of OTUs/ASVs)?  

*Mothur*   
Across all samples, there are 15 OTUs within cyanobacteria phylum. Table 3 presents the numbers of OTUs within cyanobacteria phylum for each sample. Most of samples contain  3-5 OTUs within cyanobacteria, except Saanich_120 and Saanich_200 which only have 1 and 0 cyanobacteria OTUs respectively.  

*QIIME2*    
There are 51 ASVs within cyanobacteria phylum across all samples. The number of ASVs within cyanobacteria phylum for each sample is shown in Table 2. Saanich_010, Saanich 120, and Saanich_135  have relatively high ASV number equal to or over 15. However, it is important to note that there are no singletons within the ASV dataset, and the function used to estimate richness, "estimate_richness" from the phyloseq library is highly dependent on the number of singletons, and warns of unreliable or wrong results in the absence of singletons in the data.  


######Table showing richness of OTUs using mothur data and ASVs using QIIME2 data  
```{r, echo=FALSE, "Question 3a"}
m.norm %>% 
  subset_taxa(Phylum=="Cyanobacteria")

mothur_otu=m.norm %>% subset_taxa(Phylum=="Cyanobacteria") %>% estimate_richness(measures = c("Observed"))

q.norm %>% 
  subset_taxa(Phylum=="D_1__Cyanobacteria")

qiime_asv=q.norm %>% subset_taxa(Phylum=="D_1__Cyanobacteria") %>%estimate_richness(measures = c("Observed"))

sam_data=data.frame(m.norm@sam_data)

Depth=sam_data%>%select("Depth_m")

otu_asv=cbind(Depth, mothur_otu, qiime_asv)
colnames(otu_asv)=c("Depth_m","OTU","ASV")
kable(otu_asv, caption="Table 3. OTUs/ASVs across depth",align = "c")

```

4. Do the abundances of OTUs/ASVs within your taxon of interest change significantly with depth and/or oxygen concentration?  

Using the linear model for statistical interpretation, after correcting the p-value for multiple comparisons, the abundance of  OTUs 0189, 0658, 1104, 3852, and 4312 from the mothur pipeline within the cyanobacteria phylum changed significantly with both depth and oxygen. Interestingly, the abundance of ASVs in the QIIME2 pipeline did not have any significant changes across the depth profiles after correcting for the p-value. However, there were 17 ASVs that had a significant abundance change with respect to oxygen concentrations.  

######Mothur
```{r, echo=FALSE, "Question 4a",fig.width=10,fig.height=7}
#filtered table 
m.Cyano=m.norm %>% subset_taxa(Phylum=="Cyanobacteria") 
m.filtered=data.frame(m.Cyano@tax_table)

#General linear model for each OTU across depth
otu_stats = data.frame("Estimate" = numeric(0), "Std. Error"= numeric(0),"t value"= numeric(0),"P_value"= numeric(0))
for (otu in row.names(m.filtered)){
  require(tidyverse)
  require(knitr)
    linear_fit = m.norm %>% 
    psmelt() %>% 
    filter(OTU==otu) %>% 
    lm(Abundance ~ Depth_m, .) %>% 
    summary()
  otu_data = linear_fit$coefficients["Depth_m",]
  otu_stats <- rbind(otu_stats, otu_data)
}

colnames(otu_stats)<- (c("Estimate", "Std. Error","t value","P_value"))
otu_depth=otu_stats%>%
  mutate(Adjusted_P=p.adjust(P_value, method="fdr"))
row.names(otu_depth) <- row.names(m.filtered)
otu_depth_filter=otu_depth[which(otu_depth[,"Adjusted_P"]<=0.05),]
kable(otu_depth_filter,caption="Table 4. Correlation data of Cyanobacteria OTUs with significant differences across depth using mothur data",align = "c")

#Calculating abundances of OTUs across depth
m.perc %>%
  subset_taxa(Phylum=="Cyanobacteria") %>%
  psmelt() %>%
 
ggplot() +
  geom_point(aes(x=Depth_m, y=Abundance)) +
  geom_smooth(method='lm', aes(x=Depth_m, y=Abundance)) +
  facet_wrap(~OTU, scales="free_y") +
  labs(caption="Figure 7. Abundance of OTUs within Phylum Cyanobacteria across depth using mothur data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(0.9)))

#General linear model for each OTU across oxygen
otu_stats_O2 = data.frame("Estimate" = numeric(0), "Std. Error"= numeric(0),"t value"= numeric(0),"P_value"= numeric(0))
for (otu in row.names(m.filtered)){
  require(tidyverse)
  require(knitr)
    linear_fit = m.norm %>% 
    psmelt() %>% 
    filter(OTU==otu) %>% 
    lm(Abundance ~ O2_uM, .) %>% 
    summary()
  otu_data_O2 = linear_fit$coefficients["O2_uM",]
  otu_stats_O2 <- rbind(otu_stats_O2, otu_data_O2)
}

colnames(otu_stats_O2)<- (c("Estimate", "Std. Error","t value","P_value"))
otu_oxygen=otu_stats_O2%>%
  mutate(Adjusted_P=p.adjust(P_value, method="fdr"))
row.names(otu_oxygen) <- row.names(m.filtered)
otu_oxygen_filter=otu_oxygen[which(otu_oxygen[,"Adjusted_P"]<=0.05),]
kable(otu_oxygen_filter,caption="Table 5. Correlation data of Cyanobacteria OTUs with significant differences across oxygen using mothur data", align = "c")

#Calculating abundances of OTUs across oxygen
m.perc %>%
  subset_taxa(Phylum=="Cyanobacteria") %>%
  psmelt() %>%
 
ggplot() +
  geom_point(aes(x=O2_uM, y=Abundance)) +
  geom_smooth(method='lm', aes(x=O2_uM, y=Abundance)) +
  facet_wrap(~OTU, scales="free_y") +
  labs(caption="Figure 8. Abundance of OTUs within Cyanobacteria across oxygen using mothur data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(0.9)))

```

######QIIME2
```{r, echo=FALSE, "Question 4b_1"}
#filtered table 
q.Cyano=q.norm %>% subset_taxa(Phylum=="D_1__Cyanobacteria") 
q.filtered=data.frame(q.Cyano@tax_table)

#General linear model for each ASV across depth
asv_stats = data.frame("Estimate" = numeric(0), "Std. Error"= numeric(0),"t value"= numeric(0),"P_value"= numeric(0))
for (asv in row.names(q.filtered)){
  require(tidyverse)
  require(knitr)
    linear_fit = q.norm %>% 
    psmelt() %>% 
    filter(OTU==asv) %>% 
    lm(Abundance ~ Depth_m, .) %>% 
    summary()
  asv_data = linear_fit$coefficients["Depth_m",]
  asv_stats <- rbind(asv_stats, asv_data)
}

colnames(asv_stats)<- (c("Estimate", "Std. Error","t value","P_value"))
asv_depth=asv_stats%>%
  mutate(Adjusted_P=p.adjust(P_value, method="fdr"))
row.names(asv_depth) <- row.names(q.filtered)

asv_depth_filter=asv_depth[which(asv_depth[,"Adjusted_P"]<=0.05),]
z=nrow(asv_depth_filter)

if (z!=0) {
  kable(asv_depth_filter,caption="Correlation data of Cyanobacteria ASVs with significant differences across depth using QIIME2 data",align = "c")
}else{
    print("None of ASV has significantly different abundance acrossing depth with QIIME2 data")}
```

```{r, echo=FALSE, "Question 4b_2", fig.width=15, fig.height=10}
#Calculating abundances of ASVs across depth
q.perc %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  psmelt() %>%
 
ggplot() +
  geom_point(aes(x=Depth_m, y=Abundance)) +
  geom_smooth(method='lm', aes(x=Depth_m, y=Abundance)) +
  facet_wrap(~OTU, scales="free_y") +
  labs(caption="Figure 9. Abundance of ASVs within cyanobacteria across depth using QIIME2 data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.3)))
```

```{r, echo=FALSE, "Question 4b_3"}
#General linear model for each ASV across oxygen
asv_stats_O2 = data.frame("Estimate" = numeric(0), "Std. Error"= numeric(0),"t value"= numeric(0),"P_value"= numeric(0))
for (asv in row.names(q.filtered)){
  require(tidyverse)
  require(knitr)
    linear_fit = q.norm %>% 
    psmelt() %>% 
    filter(OTU==asv) %>% 
    lm(Abundance ~ O2_uM, .) %>% 
    summary()
  asv_data_O2 = linear_fit$coefficients["O2_uM",]
  asv_stats_O2 <- rbind(asv_stats_O2, asv_data_O2)
}

colnames(asv_stats_O2)<- (c("Estimate", "Std. Error","t value","P_value"))
asv_oxygen=asv_stats_O2%>%
  mutate(Adjusted_P=p.adjust(P_value, method="fdr"))
row.names(asv_oxygen) <- row.names(q.filtered)
asv_oxygen_filter=asv_oxygen[which(asv_oxygen[,"Adjusted_P"]<=0.05),]
kable(asv_oxygen_filter,caption="Table 6. Correlation data of Cyanobacteria ASVs with significant differences across oxygen using QIIME2 data", align = "c")
```

```{r, echo=FALSE, "Question 4b_4", fig.width=16, fig.height=10}
#Calculating abundance of ASVs across oxygen
q.perc %>%
  subset_taxa(Phylum=="D_1__Cyanobacteria") %>%
  psmelt() %>%
 
ggplot() +
  geom_point(aes(x=O2_uM, y=Abundance)) +
  geom_smooth(method='lm', aes(x=O2_uM, y=Abundance)) +
  facet_wrap(~OTU, scales="free_y") +
  labs(caption="Figure 10. Abundance of ASVs within Cyanobacteria across oxygen using QIIME2 data")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.3)))
```

5. Are the answers to the above the same using mothur and QIIME2 processed data?  

In terms of differences between the two pipelines, the overall trend between both datasets were similar, however the statistical interpretations of the datasets varied. For starters, the Shannon diversity for the whole microbial community processed with mothur was generally smaller than that estimated by QIIME 2 processed data. Additionally, ANOVA tests indicated that the Shannon diversity had no significant change across depth from the mothur pipeline (p = 0.054). In contrast, there was statistical significance with the Shannon diversity across depth from the QIIME2 dataset (p = 0.022). Within the Cyanobacterial taxon, there were 51 ASVs calculated from the QIIME2 pipeline compared to 15 OTUs from the Mothur pipeline. Interestingly, while there is a difference in the richness between both datasets, 5 of the OTUs had a significantly different abundance with respect to depth and oxygen profiles, while none of the ASVs had a significant change in abundance across depth and 17 ASVs which had a significant difference in abundance across oxygen concentrations. Despite all these variations, the Cyanobacterial taxon itself had a significant difference in abundance with respect to depth and oxygen profiles along the water column across both pipelines.

###### ANOVA on alpha-diversity of mothur data across depth 
```{r,echo=FALSE}
#Mothur data - ANOVA test on Alpha diversity across depth
summary(aov(Shannon ~ Depth_m, data = m.meta.alpha))
```
###### ANOVA on alpha-diversity of QIIME2 data across depth
```{r, echo=FALSE}
#Qiime2 data - ANOVA test on Alpha diversity across depth
summary(aov(Shannon ~ Depth_m, data = q.meta.alpha))
```

####Discussion

The significant differences for Cyanobacteria abundance across depth and oxygen may be caused by photosynthesis and water temperature. Bacteria of the phylum Cyanobacteria obtain their energy through photosynthesis. These phototrophs are characterized by phycocyanin, a bluish pigment, which functions as an auxiliary light-harvesting protein complex to chlorophyll. Phycocyanin absorbs orange and red light at approximately 620nm and fluoresces at about 650nm depending on the species (10). This is particularly interesting because orange and red light are typically absorbed within the first 50m of a water column (11). From our data in Fig 5 & 6, it was found that there are significant differences in the abundance of Cyanobacteria across the Saanich Inlet depth profiles in both the mothur and QIIME2 pipelines. It is likely that this significance exists as a result of Cyanobacteria thriving at the upper boundaries of the water column, where they are still able to absorb red and orange light that is essential for Cyanobacterial photosynthesis. Furthermore, these findings are supported by the oxygen and fluorescence profiles across the water column (Fig 11). There is a significant difference in the concentration of oxygen and chlorophyll A across depth profiles. Higher concentrations of chlorophyll A and oxygen were found within the top 50m, which indicates increased photosynthetic activity. Moreover, there was a significant difference in the abundance of Cyanobacteria across oxygen concentrations and chlorophyll A concentrations for both pipelines; high oxygen and chlorophyll A concentrations were associated with a larger abundance. With Cyanobacterial photosynthesis contributing a substantial proportion of oxygen to Earth's atmosphere, it is not surprising that a high abundance of Cyanobacteria is associated with a high concentration of oxygen and chlorophyll A, and a shallow depth. Moreover, it has been reported that the growth rate of cyanobacteria is significantly influenced by temperature. Cyanobacteria were observed to have a lower growth rate at colder temperatures. For marine cyanobacteria, the optimal growth temperature range is 20 - 27.5^o^C; at these temperatures, cyanobacteria grow at a rate of 0.8 d^-1^  (12). When the temperature dropped to approximately 15^o^C, the growth rate of cyanobacteria slowed to 0.22 d^-1^. Interestingly, when temperature dropped below 10^o^C, cyanobacterial growth rates came to a complete stop (13). Therefore, in our study, the decline of cyanobacteria abundance with depth may at least partly attributed to the decreasing temperature. According temperature data for each sample, the temperature is close to 13^o^C at 10m, and decreases to about 9^o^C when at depths below 100m. Hence, the growth rate of cyanobacteria at depths below 100m is substantially slower than the growth rate at the water's surface. This leads to a lower abundance at lower depths.  

Implications of potential differences in pipelines for microbial ecology make it difficult to make conclusive statements in research and discovery, since we become unable to differentiate actual biological differences seen and differences due to a particular pipeline being used. This could also suggest that one pipeline is more suited to the dataset. In fact, this difference could also be exploited, and manipulated so that a pipeline is selected based on the results that it gives, rather than the more appropriate pipeline for the given dataset.  

In the context of this project, the main difference between the two pipelines is based on whether the pipeline produces OTUs (mothur) or ASVs (QIIME2). Both of them use different clustering algorithms to determine "true" sequences, and as a result, there are usually far more ASVs in the QIIME2 pipeline than OTUs created with the mothur pipeline. Therefore, when doing downstream analysis of the pipeline results, this may be one of the reasons why there is a large disparity in between the numbers of sequences and quality of the sequences produced even when using the same initial data.  

However, when it came to counting abundances within our taxon, cyanobacteria, there was a far lower abundance seen in qiime2 results than mothur results which cannot be explained by having more numerous ASVs than OTUs. Interestingly, when running the estimate_richness function on our qiime2 data for determining abundance, this resulted in the warning that our data provided did not have any singletons (supposedly in the output), and that results are probably unreliable or wrong. This error did not occur with running this function on mothur data.
Further analysis of why this error is seen with qiime2 data should be done before fully trusting the results of this function with qiime2 data.  

In subsequent analyses with these two pipelines, perhaps a more in-depth analysis of each function available with the phyloseq package should be tested. A dataset of a well-studied and known community could be used so that the output of the functions can be compared with respect to the pipeline used. Comparisons can be made for the outputs of each pipeline, and results from the mothur and QIIME2 pipelines can be assessed with reference to the expected results. Use cases should also be considered, and standard use cases for either pipeline should be indicated so exploitation of pipelines for favourable results does not occur. Mothur may be more suited to a particular dataset, while QIIME2 could be more appropriate for another dataset that should be used with a "denoising" algorithm.  

Future directions for this project could possibly involve the analysis of water samples from other OMZs at various depths, along with observation of the Cyanobacteria data present in those samples to see if the relationships are exhibited between oxygen, nitrogen, and the phyla population as with this study. These can also be analyzed once again with both mothur and QIIME2 for further comparison between the pipelines.  

```{r, echo=FALSE, fig.width=10, fig.height=7}
# Geochemical parameters across depth
sam_data=data.frame(m.norm@sam_data)
sam_data1=sam_data%>%mutate(CH4_uM=CH4_nM/1000)
nutrient=sam_data1 %>% select(Depth_m,O2_uM,PO4_uM,SiO2_uM,NO3_uM,NH4_uM,NO2_uM,H2S_uM,N2O_nM,CH4_uM,Temperature_C, Fluorescence_mgm_3) %>% gather(nutrient,uM,O2_uM,PO4_uM,SiO2_uM,NO3_uM,NH4_uM,NO2_uM,H2S_uM,N2O_nM,CH4_uM, Temperature_C, Fluorescence_mgm_3)
library(ggplot2)
ggplot(nutrient,aes(x=Depth_m,y=uM))+
  geom_line()+
  geom_point()+
  ylab("Concentration or degree or mgm3")+
  facet_wrap(~nutrient,scales="free_y")+
  labs(caption="Figure 11. Geochemical parameters across depth")+
  theme(plot.caption = element_text(hjust=0.5, size=rel(0.8)))
```

####References

1. Walsh DA, Zaikova E, Howes CG, Song YC, Wright JJ, Tringe SG, Hallam SJ. 2009. Metagenome of a versatile chemolithoautotroph from expanding oceanic dead zones. Science 326(5952): 578-582.  
2. Torres-Beltr?n M, Hawley AK, Capelle D, Zaikova E, Walsh DA., Mueller A, Finke J. 2017. A compendium of geochemical information from the Saanich Inlet water column. Nature scientific data 4(170159).  
3. Hawley AK, Torres-Beltr?n M, Zaikova E, Walsh DA, Mueller A, Scofield M, Kheirandish S, Payne C, Pakhomova L, Bhatia M, Shevchek O, Gies EA, Fairley D, Malfatii SA, Norbeck AD, Brewer HM, Pasa-Tolic, L, del Rio TG, Suttle CA, Trige S, Hallam SJ. Data Descriptor: A compendium of multi-omic sequence information from the Saanich Inlet water column. Nature scientific data 4(170160).  
4. Hallam SJ, Torres-Beltr?n M, Hawley AK. Comment: Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Nature scientific data 4(170158).  
5. National Research Council. 2007. The new science of metagenomics: revealing the secrets of our microbial planet. National Academies Press, Washington, DC.  
6. Bowers RM, Kyrpides NC, Stepanauskas R, Harmon-Smith M, Doud D, Reddy TBK, Tringe SG. 2017. Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. Nature Biotechnology 35: 725-731.  
7. Falkowski PG, Fenchel T, Delong EF. 2008. The microbial engines that drive Earth's biogeochemical cycles. Science 320(5879): 1034-1039.  
8. Wooley JC, Godzik A, Friedberg I. 2010. A primer on metagenomics. PLoS computational biology 6(2), e1000667.  
9. Callahan BJ, McMurdie PJ, Holmes SP. 2017. Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. The ISME journal 11(12): 2639-2643.  
10. Simis SG, Huot Y, Babin M, Seppala J, and Metsamaa L. 2012. Optimization of variable fluorescence measurements of phytoplankton communities with cyanobacteria. Photosynthesis research 112(1): 13-30.  
11. Light in the ocean (n.d.) [Online]. Link: https://manoa.hawaii.edu/exploringourfluidearth/physical/ocean-depths/light-ocean.  
12. Boyd PW,  RynearsonTA, Armstrong EA, Fu F, Hayashi K, Hu Z, Hutchins DA, Kudela RM, Litchman E, Mulholland MR, Passow U, Strzepek RF,  Whittaker KA,  Yu E,  and Thomas MK. (2013). Marine phytoplankton temperature versus growth responses from polar to tropical waters - outcome of a scientific community-wide study. PLoS ONE 8(5): e63091.  
13. Berg M, Sutula M. 2015. Factors affecting the growth of cyanobacteria with special emphasis on the Sacramento-San Joaquin Delta. Southern California Coastal Water Research Project Technical Report 869.  

### Project_01_Self-assessment

To facilitate group assessment, please indicate your contributions to the following aspects of your final group reports. You should provide short statements under the headers relevant to your work. For example, you could specify that you "wrote the first draft", "wrote the code to accomplish X figure or result", "troubleshot code", "interpreted results", etc.

Include **separate** completed versions of this document for Project 1 **and** Project 2 in your portfolio repo; they do not need to be incorporated into the final knit portfolio html.

#### Abstract  
##### Editing  
I edited the format of abstract. 

#### Results  
##### Anaysis   
I wrote the code of statistical analysis to answer question 1, 3, 4, and 5.

##### Figures  
I wrote the code of the figures to answer question 1, 3, and 4. 

##### Writing  
I wrote the first draft for question 2, 3, and 5.  

##### Editing  
I troubleshot the code for question 1, 3, and 4 and figure formats and edited the answer to the section of taxa presence and abundance in question 1.  

#### Discussion  
##### Literature research  
I looked at literatures on the effects of geochemical parameters on the distribution of Cyanobacteria across depth and oxygen.

##### Writing  
I wrote the effect of temperature on Cyanobcateria distribution. 

##### Analysis  
I wrote the code of the figure regarding the distribution of geochemical parameters across depth.

##### Editing  
I edited the part of the impacts of greochemical parameters on Cyanobacteria distribution. 

# Module_04: Distributed metabolic networks

## Project_02

### Project_02_Report

####Authors
Team 1: Albert Chang (), Alison Fong (33399149), Karen Lau (16524143), Yaqian
  Luo (59751503), Jessica Ngo (31837131), Peter (Kiet) Truong (36645133)

####Abstract  

Genes encoding key steps in the nitrogen cycle are well defined and provide a basis for functional anchor screening to determine their distribution across prokaryotic taxa. In this study, Saanich Inlet was used as a model ecosystem to investigate the metabolic coupling and symbiotic interactions that influence the nitrogen cycle in oxygen minimum zones. Water samples were collected at seven major depths spanning the oxycline, and genomic DNA and RNA were extracted and sequenced. The resulting reads were processed, assembled, and analyzed using the Tree-based Sensitive and Accurate Protein Profiler (TreeSAPP) pipeline to reconstruct the nitrogen cycle along defined redox gradients in Saanich Inlet. The *narG* gene was then investigated in detail. Its DNA levels were found to increase with depth while RNA levels decreased with depth. Proteobacteria contributed most *narG* DNA, while Actinobacteria contributed most RNA. Further, *narG* RNA abundance increased along with nitrite concentrations in the Inlet, but had the opposite relationship with nitrate concentrations; this was the expected result given that *narG* mediates the conversion of nitrate to nitrite. Overall, *narG* is responsible for a conversion taking place the beginning of the nitrogen cycle, and this provides reasoning for its presence at the given depths, while providing a key piece to exemplify evolutionary and environmental reasoning for the distributed metabolism seen in the nitrogen cycle.

####Introduction  

Oxygen minimum zones (OMZs) are regions in the ocean where dissolved oxygen concentrations fall below 20 $\mu$M (1). Because of temperature increases and other effects caused by global warming, OMZs are rising at a notable rate. Saanich Inlet is a seasonally anoxic fjord off the coast of British Columbia and a model ecosystem for studying the diversity and biochemical responses of microbial communities to the hypoxic environments commonly observed in OMZs (1, 2). In particular, Saanich Inlet has been used to model the metabolic coupling and symbiotic interactions that occur in OMZs (3).  The inlet undergoes recurring cycles of water column stratification and deep water renewal, making it a model ecosystem for studying microbial responses to changes in oceanic oxygenation levels (4). Increased levels of primary productivity in ocean surfaces during the spring season, along with the limited mixing which occurs between the basin and surface waters both result in the development of an anoxic body of water with increasing depth in the Inlet (2). These anoxic regions become highly populated with chemolithoautotrophs, eventually leading to a decrease in aerobically respiring organisms found deeper within these zones. Past studies have shown that these types of metabolic shifts usually lead to a loss of nitrogen along with the production of notable greenhouse gases such as methane (CH~4~) and nitrous oxide (N~2~O) (1). Some species have also been shown to engage in sulfide (H~2~S) oxidation and nitrate (NO~3~^-^) reduction pathways within these zones (5). 

The nitrogen cycle--the biogeochemical cycle by which nitrogenous compounds are interconverted between chemical forms for environmental circulation--consists mainly of nitrogen fixation, nitrification and denitrification, and has been catalyzed by microorganisms long before the appearance of humans (6). Even today, much of it is still dictated by the diverse microbial niches present in the surrounding environment. An example of this control can be seen in denitrification--the conversion of NO~3~^-^ to nitrogen gas (N~2~)--carried out by denitrifying bacteria. Denitrication is a highly important process as it prevents the accumulation of nitrogen compounds to toxic levels that could lead to eutrophication, and maintains the homeostasis of nitrogen distribution between soil and atmosphere. Having an atmospheric residence time of approximately 1 billion years, nitrogen gas (N~2~) is highly inert and is not accessible for the synthesis of proteins and nucleic acids, though this problem is easily solved by the conversion of N~2~ to NH~4~^+^ through nitrogen fixation (6).  

The aim of our project was to characterize the involvement of our gene of interest, *narG*, which is known for its role in the reduction of NO~3~^-^ to NO~2~^-^ in the denitrification cycle. Recent literature has highlighted that microbial taxa do not necessarily integrate whole elemental cycles into their individual metabolic pathways, but often divide these reactions among the community so that consortia of taxa must rely on each other to fulfill their metabolic requirements (2).  
  

####Methods  

Water samples were collected from Saanich Inlet at depths 10, 100, 120, 135, 150, 165, and 200m spanning the oxycline during Saanich Inlet Cruise 72. The samples were subsequently filtered through a 0.22 $\mu$m Sterivex filter to collect biomass and both genomic DNA and RNA were extracted from these samples. The extracted RNA was converted into cDNA. Both total genomic DNA and cDNA were used to construct the shotgun Illumina libraries. Sequencing data were generated on the Illumina HiSeq platform with 2x150bp technology. Further sampling and sequencing details can be found in *A compendium of multi-omic sequence information from the Saanich Inlet water column* by Hawkley et al. (2017).  

The IMG/M pipeline was then used to process the resulting reads. Metapathways 2.5 was used to assemble and process the metagenomes. A beta version of Tree-based Sensitive and Accurate Protein Profiler (TreeSAPP) pipeline was used to reconstruct the nitrogen cycle along defined redox gradients in Saanich Inlet using Google Cloud services. iTOL version 4.0 was used to generate phylogenetic trees based on the processed data. The contig maps were loaded into R version 3.4.3. Tidyverse, cowplot, phyloseq, and kableExtra packages were loaded and used to complete the data analysis.  


####Results  

#####Environment setup and Data Cleaning  
```{r echo=FALSE}
library(tidyverse)
library(cowplot)
library(phyloseq)
library(kableExtra)
```

Loading, parsing (initial data contains data for all genes - we are only interested in *narG*), renaming data.  We only need taxonomy, abundance, and query data.
```{r message=FALSE, echo=FALSE}
narG.DNA.10m = read_tsv("narG/marker_contig/DNA10m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%
    select(Tax.DNA.10 = Confident_Taxonomy, Abund.DNA.10 = Abundance, Query)

narG.RNA.10m = read_tsv("narG/marker_contig/RNA10m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>% 
    select(Tax.RNA.10 = Confident_Taxonomy, Abund.RNA.10 = Abundance, Query)

narG.DNA.100m = read_tsv("narG/marker_contig/DNA100m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>% 
    select(Tax.DNA.100 = Confident_Taxonomy, Abund.DNA.100 = Abundance, Query)

narG.RNA.100m = read_tsv("narG/marker_contig/RNA100m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.RNA.100 = Confident_Taxonomy, Abund.RNA.100 = Abundance, Query)

narG.DNA.120m = read_tsv("narG/marker_contig/DNA120m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.DNA.120 = Confident_Taxonomy, Abund.DNA.120 = Abundance, Query)

narG.RNA.120m = read_tsv("narG/marker_contig/RNA120m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.RNA.120 = Confident_Taxonomy, Abund.RNA.120 = Abundance, Query)

narG.DNA.135m = read_tsv("narG/marker_contig/DNA135m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.DNA.135 = Confident_Taxonomy, Abund.DNA.135 = Abundance, Query)

narG.RNA.135m = read_tsv("narG/marker_contig/RNA135m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.RNA.135 = Confident_Taxonomy, Abund.RNA.135 = Abundance, Query)

narG.DNA.150m = read_tsv("narG/marker_contig/DNA150m_marker_contig_map.tsv") %>%
    filter(Marker=="narG") %>%  
    select(Tax.DNA.150 = Confident_Taxonomy, Abund.DNA.150 = Abundance, Query)

narG.RNA.150m = read_tsv("narG/marker_contig/RNA150m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.RNA.150 = Confident_Taxonomy, Abund.RNA.150 = Abundance, Query)

narG.DNA.165m = read_tsv("narG/marker_contig/DNA165m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.DNA.165 = Confident_Taxonomy, Abund.DNA.165 = Abundance, Query)

narG.RNA.165m = read_tsv("narG/marker_contig/RNA165m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.RNA.165 = Confident_Taxonomy, Abund.RNA.165 = Abundance, Query)

narG.DNA.200m = read_tsv("narG/marker_contig/DNA200m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.DNA.200 = Confident_Taxonomy, Abund.DNA.200 = Abundance, Query)

narG.RNA.200m = read_tsv("narG/marker_contig/RNA200m_marker_contig_map.tsv") %>% 
    filter(Marker=="narG") %>% 
    select(Tax.RNA.200 = Confident_Taxonomy, Abund.RNA.200 = Abundance, Query)
```

Data manipulation into a single data frame
```{r, echo=FALSE}
narG.all = narG.DNA.10m %>% 
# Combine the data frames will full_join to keep all the data
  full_join(narG.RNA.10m, by = "Query") %>%
  full_join(narG.DNA.100m, by = "Query") %>% 
  full_join(narG.RNA.100m, by = "Query") %>%
  full_join(narG.DNA.120m, by = "Query") %>% 
  full_join(narG.RNA.120m, by = "Query") %>%
  full_join(narG.DNA.135m, by = "Query") %>% 
  full_join(narG.RNA.135m, by = "Query") %>%
  full_join(narG.DNA.150m, by = "Query") %>% 
  full_join(narG.RNA.150m, by = "Query") %>%
  full_join(narG.DNA.165m, by = "Query") %>% 
  full_join(narG.RNA.165m, by = "Query") %>% 
  full_join(narG.DNA.200m, by = "Query") %>% 
  full_join(narG.RNA.200m, by = "Query") %>% 
# Create a taxonomy variable aggregating all taxonomy columns so as to fill in any NAs that might occur. !is.na means "is not NA", so the following says that the Taxonomy data should be taken from Tax.RNA.10 if that is not NA, or else take it from Tax.DNA.10 if that is not NA, or else Tax.RNA.200, etc. until if all are NA, give Taxonomy of "unclassified"
  mutate(Taxonomy = ifelse(!is.na(Tax.RNA.10), Tax.RNA.10,
                    ifelse(!is.na(Tax.DNA.10), Tax.DNA.10,
                    ifelse(!is.na(Tax.RNA.100), Tax.RNA.100,
                    ifelse(!is.na(Tax.DNA.100), Tax.DNA.100,
                    ifelse(!is.na(Tax.RNA.120), Tax.RNA.120,
                    ifelse(!is.na(Tax.DNA.120), Tax.DNA.120,
                    ifelse(!is.na(Tax.RNA.135), Tax.RNA.135,
                    ifelse(!is.na(Tax.DNA.135), Tax.DNA.135,
                    ifelse(!is.na(Tax.RNA.150), Tax.RNA.150,
                    ifelse(!is.na(Tax.DNA.150), Tax.DNA.150,
                    ifelse(!is.na(Tax.RNA.165), Tax.RNA.165,
                    ifelse(!is.na(Tax.DNA.165), Tax.DNA.165,
                    ifelse(!is.na(Tax.RNA.200), Tax.RNA.200,
                    ifelse(!is.na(Tax.DNA.200), Tax.DNA.200,
                           "unclassified"))))))))))))))) %>% 
# Get rid of the old Tax variables
  select(-starts_with("Tax.")) %>% 
# Gather all the abundance data into 1 column 
  gather("Key", "Abundance", starts_with("Abund")) %>% 
# Turn the Key into Depth and RNA/DNA variables. We can easily do this because we specifically named these variables with period separation when we loaded in the original data
  separate(Key, c("Key","Type","Depth_m"), by = ".") %>% 
# Remove Key variable now that it only contains "Abund". This also serves to reorder the columns so that the very long Query is at the end.
  select(Depth_m, Type, Abundance, Taxonomy, Query) %>% 
# Make sure R knows depth is numerical since it came from a character variable
  mutate(Depth_m = as.numeric(Depth_m)) %>% 
# Separate Taxonomy into columns so we can plot at different levels
  separate(Taxonomy, into = c("Domain", "Phylum", "Class", "Order", "Family", "Genus", "Species"), sep="; ")
```

Warning above ignored, because not all queries could be classified down to species level, so these cells are filled as NA, which is fine.

##### Final data table
```{r, echo=FALSE}
kable(narG.all,"html", caption="Table 1. Final data of narG gene")%>%
  kable_styling() %>%
  scroll_box(width = "1000px", height = "600px")
```

##### 1. How does the DNA abundance of *narG* gene differ with depth?  

Figure 1 indicates that the DNA abundance of the *narG* gene has an overall increasing trend with depth. The DNA abundance of the *narG* gene increases gradually from 21.40 to 990.72 at depth from 10 m to 150 m. However, it declines to 249.21 at depth of 165 m, but increases again to 717.90 at depth of 200 m.  

```{r, echo=FALSE}
narG.all %>% 
# Filter the DNA data
  filter(Type == "DNA") %>% 
# Change NAs to "unclassified" at the level you want to plot. Here we will do Genus
  mutate(Genus = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  group_by(Depth_m) %>%
  mutate(abund_sum=sum(Abundance, na.rm=TRUE))%>%
  group_by(Depth_m)%>%
  summarize(abund_sum=mean(abund_sum))%>%

  
ggplot(aes(x = "narG", y = Depth_m)) +
# Use the size aesthetic to show abundance
  geom_point(aes(size = abund_sum)) +
# Reverse the why axis so depth increases going down
  scale_y_reverse(lim=c(200,0)) +
  labs(y="Depth (m)")+
  theme_classic() +
  scale_size_continuous(name="Abundance",breaks = c(200,400,600,800)) +
  geom_text(aes(label=abund_sum, hjust=1.5))+
  labs(caption = "Figure 1. Abundance of the narG gene (DNA) at different depths",
       x = "") +
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.0)))
```
 

##### 2. What taxa are responsible for *narG*? Are they the same with depth? With DNA versus RNA?  

RNA abundance of the *narG* gene with depth does not match the trend of DNA abundance. Figure 2 shows that RNA abundance of the *narG* gene increases significantly from 10 m to 100 m, and reaches its second highest level of 3585.45 at 100 m. Interestingly, the RNA abundance fluctuates between a range of 2500-3700 at depths 100-135 m. After that, it decreases gradually until 200 m. On the other hand, RNA abundance of *narG* gene is much larger than the DNA abundance at depth of 100-165 m, while less *narG* RNA is found in the samples from depth of 10 m and 200 m.  
```{r, echo=FALSE}
# DNA and RNA abundance with depth
DNA.abund.depth=narG.all %>% 
  filter(Type == "DNA") %>% 
  mutate(Genus = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  group_by(Depth_m) %>%
  mutate(abund_sum=sum(Abundance, na.rm=TRUE))%>%
  group_by(Depth_m)%>%
  summarize(DNA=mean(abund_sum))

RNA.abund.depth=narG.all %>% 
  filter(Type == "RNA") %>% 
  mutate(Genus = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  group_by(Depth_m) %>%
  mutate(abund_sum=sum(Abundance, na.rm=TRUE))%>%
  group_by(Depth_m)%>%
  summarize(RNA=mean(abund_sum))

left_join(DNA.abund.depth,RNA.abund.depth,by="Depth_m")%>%
  gather(Type,abund_sum,DNA:RNA)%>%

# Show both RNA and DNA using an x variable  
ggplot(aes(x = Type, y = Depth_m)) +
  geom_point(aes(size = abund_sum)) +
  labs(y="Depth (m)")+
  scale_y_reverse(lim=c(200,0)) +
  theme_classic()+
  scale_size_continuous(name="Abundance",breaks = c(500,1000,1500,2000,3000)) +
  geom_text(mapping=aes(label=abund_sum),hjust=1.2)+
  labs(caption = "Figure 2. Abundance of the narG gene (DNA vs. RNA) at different depths") +
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.0)))
```

##### 3. What taxa are responsible for *narG* gene? Are they the same with depth? With DNA versus RNA?  

As shown in Figure 3, Actinobacteria, candidatus Omnitrophica, Chlorobi, Euryarchaeota, metagenomes, Proteobacteria, and unclassified Bacteria are responsible for *narG* DNA. Among of them, Actinobacteria, candidatus Omnitrophica, Chlorobi, Euryarchaeota, and metagenomes have evenly distributed DNA abundance across depth above 100 m. However, the distribution of Proteobacterial DNA abundance with depth follows a similar trend with that of *narG* DNA abundance, in which abundance increases from depth of 10 m to 150 m, declines at depth of 150-165 m, and finally increases again at 200 m. On the contrary, DNA abundance of unclassified Bacteria is the same at depth below 165 m but increases slightly at 200 m.  

Actinobacteria, Proteobacteria, and unclassified Bacteria make dominant contributions to *narG* RNA among the 11 phyla. Actinobacteria is the predominant phylum for *narG* RNA contribution at depth of 100 m and 120 m, but its RNA abundance decreases gradually at deeper water and eventually becomes undetectable at depth of 200 m. Both Proteobacteria and unclassified Bacteria have observed RNA at the seven different depth, although their RNA abundances are not distributed in the same trend with depth. Proteobacterial RNA abundances at depth of 150 m and 200 m are higher than that from other depths, while unclassified Bacteria have the lowest RNA abundance at 150 m and 200 m. Additionally, unclassified Bacteria have RNA abundance increasing with depth at shallow water and reaching the highest point at depth of 135 m.


```{r fig.width=8, echo=FALSE}
narG.all %>% 
# Change NAs to "unclassified" at the level you want to plot
  mutate(Phylum = ifelse(is.na(Phylum), "unclassified Bacteria", Phylum)) %>% 
  
ggplot(aes(x = Phylum, y = Depth_m)) +
# Use an ifelse statement to make 0 values into NAs so that they don't show up on the plot
# Use position_dodge to keep points from overlapping
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance), color = Type), position = position_dodge(0.5)) +
  scale_y_reverse(lim=c(200,0)) +
  labs(y="Depth (m)")+
  theme_classic() +
# Rename legend
  scale_size_continuous(name = "Abundance") +  
  theme(axis.text.x = element_text(angle = 45, hjust =1, size=rel(1.2)))+
  labs(caption = "Figure 3. Abundance of phyla with narG (DNA vs. RNA) at different depths") +
  theme(plot.caption = element_text(hjust=0.5, size=rel(1.1)))
 
```


##### 4. How does the abundance of *narG* gene relate to nitrogen species in Saanich?  

The *narG* gene has significant impacts on nitrite (NO~2~^-^) and nitrate (NO~3~^-^) concentration across depth in Saanich. It is responsible for the reduction of nitrate to nitrite. As a result, when the RNA abundance of *narG* gene remains at a high level at depths 100-135m (Figure 2), nitrate concentration stops increasing and declines dramatically from its highest point at depth of 100 m (Figure 4D). On the other hand, due to poor nitrification in deeper water, although RNA of *narG* gene is less abundant at depth above 150 m, nitrate concentration continues decreasing with depth until to 0 at depth of 165 m. Similarly, *narG* is also one of the key factors for the fluctuation of nitrite concentration at depth 100-165 m. The low ammonium concentration in depth of 10-135 m indicates a high level of nitrification in the shallow water (Figure 4A), which is responsible for the significant increase of nitrate and the decrease of nitrite at depth 10-100 m. However, nitrite concentration does not keep decreasing with depth above 100m, instead, it remains around 0.09 $\mu$M as a result of high nitrate reduction level caused by the abundant RNA of *narG* gene. Therefore, *narG* gene abundance is closely related to nitrite and nitrate concentration in Saanich.  

```{r fig.width=7, fig.height=7, echo=FALSE}
load("mothur_phyloseq.RData")

metadata = data.frame(mothur@sam_data)

# DNA/RNA abundance with depth
plot1 = narG.all %>% 
# Change NAs to "unclassified" at the level you want to plot
  mutate(Phylum = ifelse(is.na(Phylum), "unclassified", Phylum)) %>% 
  
ggplot(aes(x = Phylum, y = Depth_m)) +
  geom_point(aes(size = ifelse(Abundance == 0, NA, Abundance), color = Type), position = position_dodge(0.5)) +
  scale_y_reverse(lim=c(200,0)) +
  labs(y = "Depth (m)") +
  theme_classic() +
  scale_size_continuous(name = "Abundance")+ 
  theme(axis.text.x = element_text(angle = 45, hjust =1))

#Then, we make our plots for NO2, "plot2", and NO3, "plot3"

plot2 = metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = NO2_uM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "NO2 (uM)")
plot3 = metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = NO3_uM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "NO3 (uM)")

plot4 = metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = NH4_uM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "NH4 (uM)")

plot5 = metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = N2O_nM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "N2O (nM)")

plot_grid(plot4, plot5, plot2, plot3, labels=c("A", "B", "C", "D"), axis="l", nrow=2, ncol=2, rel_widths=c(1/2, 1/2, 1/2,1/2))+
  labs(caption = "Figure 4. Nitrogen species in Saanich across depth. 
A) Ammonium (NH4) concentration across depth   B) Nitrous oxide (N2O) concentration across depth 
C) Nitrite (NO2) concentration across depth     D) Nitrate (NO3) concentration across depth") +
  theme(plot.caption = element_text(hjust=0.5, size=rel(0.75)))
```

####Discussion  

In anthropogenic times, NO~3~^-^ levels have significantly increased in environmental water systems resulting from agricultural activities (7). With increases in NO~3~^-^ entering these systems, the accumulation of nitrogen compounds above threshold levels can lead to eutrophication. Microbe-driven processes such as denitrification, are able to counteract this phenomenon by converting NO~3~^-^ to N~2~. In this context, our gene of interest, *narG*, specifically mediates the first step of this pathway. As the alpha subunit and catalytic domain of the membrane-bound dissimilatory nitrate reductase, *narG* is responsible for the conversion of NO~3~^-^ to NO~2~^-^. NO~2~^-^ will become the substrate of *nirK* and *nirS*, and through subsequent steps and enzymes, eventually lead to the production of N~2~ (8). Interestingly, it has been found that the process of transporting nitrate into the cytoplasmic location of the active site of NarG is inhibited by oxygen, making denitrification an anaerobic process (9). As a result, it is expected that there is greater amounts of *narG* transcription in OMZs compared to surface level waters. From figure 2, peak transcription of *narG* is found at 100 m where oxygen levels are approaching OMZ conditions (Figure 5). Additionally, the large abundance of *narG* transcription at 100 m and deeper can be explained with the corresponding abundance of NO~3~^-^ substrate. Many denitrifying facultative anaerobic microorganisms respire NO~3~^-^ in the absence of oxygen. Denitrification is considered to be the highest energy-yielding respiration system in anoxic environments (10). Increases in NO~3~^-^ at anoxic depth leads to the reduction of nitrate to nitrite in the cytoplasm by *narG*. It has been found that this process is coupled to the translocation of protons into the periplasm which directly contributes to a proton-motive force for energy conservation (8). Therefore, it is not surprising that *narG* transcription levels are near absent at oxic depths where oxygen can still be used as the preferential terminal electron acceptor (TEA), and abundant at anoxic depths where NO~3~^-^ can be used as an alternative TEA.  

```{r fig.width=5, fig.height=5, echo=FALSE}
# Oxygen with depth
metadata %>% 
# Order the data by depth  
  arrange(Depth_m) %>% 
  
ggplot(aes(x = O2_uM, y = Depth_m)) +
    geom_point() +
    geom_path(aes(group = 1)) +
    scale_y_reverse(lim=c(200,0)) +
  theme_classic() +
  labs(y = "Depth (m)",
       x = "O2 (uM)")+
  labs(caption="Figure 5. Oxygen in Saanich across depth")+
  theme(plot.caption = element_text(hjust=0.5,size=rel(1.0)))
```

Metabolisms in the nitrogen cycle is widely distributed, such that each reaction step is catalyzed by a specific enzyme from a different bacterial species carrying the corresponding genes. This distribution may be the consequence of the distinct energy and nutrient sources which are available in different ecosystems, as well as other mechanisms, namely horizontal gene transfer, and adaptive gene loss.  

In aerobic environments like shallow water and topsoil, nitrification is the predominant process of the nitrogen cycle, due to the availability of ammonia and nitrite as energy sources for nitrifying bacteria and the inhibitive effect of oxygen on denitrifiers (11,12). On the contrary, denitrification occurs in anoxic environments including OMZs and marine sediments, where nitrate serves as energy and nutrient source and the limitation of oxygen on denitrification is minimum (8). Besides, some bacteria evolve additional mechanisms to protect nitrogen-related enzymes from unfavorable environments. For example, *Trichodesmium* spp. and *Nodularia* spp. adopt the strategy that nitrogen fixation is separated from photosynthesis, in order to minimize oxygen suppression on nitrogenases, an enzyme for nitrogen fixation (8). Therefore, environmental conditions is one of the possible reason for the distributed metabolisms.  

Another potential explanation for metabolic distribution of the nitrogen cycle is horizontal gene transfer (HGT). Bacteria may acquire nitrogen-related genes from other species through HGT and take up the niche of corresponding reactions in the nitrogen cycle. For instance, ammonia monooxygenase genes, encoding the key enzyme required for ammonia oxidation, are observed widely distributed among different bacterial species (6). As a result, if a certain bacterial species responsible for ammonia oxidation became extinct in a specific ecosystem, other species, carrying ammonia monooxygenase genes and able to perform the process of ammonia oxidation, would take up the niche of the extinct one.  On the other hand, the retention of horizontally transferred gene is largely driven by selective pressure on nutrients and energy, and thus different processes in the nitrogen cycle are widely distributed among microorganisms and ecosystems (6).   

Gene loss may also make contributions to the wide distribution of metabolisms in the nitrogen cycle. A hypothesis, called as Black Queen Hypothesis (BQH), is proposed recently to explain the dependency of free-living microorganisms having metabolic gene loss on other species (13). In this hypothesis, two groups of bacterial species are defined: "helper" and "beneficiaries". The hypothesis predicts that in a microbial community, "beneficiaries", which is selected to lose a costly and dispensable functions, will obtain the function from a fraction of other individuals in the community, until the function is sufficient to support the whole community (13). Based on the hypothesis, it can be explained that why nitrogen fixation is performed by a small fraction of microorganisms in the ocean. The possible reasons may be that genes responsible for nitrogen fixation were selectively lost during evolution and fixed N was turned into a public good (13).  

This study paves the way for future investigations into metabolic coupling and symbiotic interactions in the nitrogen cycle. It appears that *narG* levels are regulated by oxygen levels and availability of the NO~3~^-^ substrate; however, the effect of other environmental factors has yet to be related with gene abundance. According to Palmer et al., in sediment communities, Actinobacterial  *narG* dominates in extreme environments characterized by low pH or high temperature, while Proteobacterial *narG* is present in pH-neutral soils (15). Given that Actinobacteria and Proteobacteria dominated DNA and transcript expression of *narG* in an aquatic context, it would be useful to determine whether these abundances are due to pH, temperature, or other measurable geochemical parameters.  

Additionally, as *narG* is inhibited by its own product, nitrite, it may be interesting to see how *narG* abundances vary with *nirK* and *nirS* abundances, which consume nitrite. Interestingly, previous studies have noted that *nirS* was dominated by $\beta$-Proteobacteria, while *nirK* was dominated by $\alpha$-Proteobacteria (14). Our study results also suggest *narG* is dominated in part by Proteobacteria, although more in depth analysis into different classes of Proteobacteria have not been attempted. A deeper analysis into the abundance of *narG* in different classes of Proteobacteria may be necessary to establish a correlation between how the abundances of *nirK*, *nirS*, and *narG* vary within the Proteobacteria phylum.  

####References  

1. Walsh DA, Zaikova E, Howes CG, Song YC, Wright JJ, Tringe SG, Hallam SJ. 2009. Metagenome of a versatile chemolithoautotroph from expanding oceanic dead zones. Science 326(5952): 578-582.  
2. Torres-Beltr?n M, Hawley AK, Capelle D, Zaikova E, Walsh DA., Mueller A, Finke J. 2017. A compendium of geochemical information from the Saanich Inlet water column. Nature scientific data 4(170159).  
3. Hawley AK, Torres-Beltr?n M, Zaikova E, Walsh DA, Mueller A, Scofield M, Kheirandish S, Payne C, Pakhomova L, Bhatia M, Shevchek O, Gies EA, Fairley D, Malfatii SA, Norbeck AD, Brewer HM, Pasa-Tolic, L, del Rio TG, Suttle CA, Trige S, Hallam SJ. Data Descriptor: A compendium of multi-omic sequence information from the Saanich Inlet water column. Nature scientific data 4(170160).  
4. Hallam SJ, Torres-Beltr?n M, Hawley AK. Comment: Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Nature scientific data 4(170158).  
5. National Research Council. 2007. The new science of metagenomics: revealing the secrets of our microbial planet. National Academies Press, Washington, DC.  
6. Falkowski PG, Fenchel T, Delong EF. 2008. The microbial engines that drive Earth's biogeochemical cycles. Science 320(5879): 1034-1039.  
7. Smith CJ, Nedwell DB, Dong LF, Osborn AM. 2007. Diversity and Abundance of Nitrate Reductase Genes (narG and napA), Nitrite Reductase Genes (nirS and nrfA), and Their Transcripts in Estuarine Sediments. Applied and environmental microbiology, 73(11): 3612-3622.  
8. Kuypers MM., Marchant, HK,  Kartal B. (2018). The microbial nitrogen-cycling network. Nature Reviews Microbiology.  
9. Moreno-Vivi?n C, Cabello P, Mart?nez-Luque M, Blasco R, Castillo F. (1999). Prokaryotic nitrate reduction: molecular properties and functional distinction among bacterial nitrate reductases. Journal of bacteriology, 181(21): 6573-6584.   
10. Strohm TO, Griffin B, Zumft WG, Schink B. (2007). Growth yields in bacterial denitrification and nitrate ammonification. Applied and environmental microbiology, 73(5): 1420-1424.  
11. Ward BB. 2008. Nitrification,  p 2511-2518. In J?rgensen SE and Fath BD(ed), Ecological Processes, vol. 3 of Encyclopedia of Ecology, 5 vols. Oxford: Elsevier.  
12. Nakajima M, Hayamize T, Nishimura H. 1983. Effect of oxygen concentration on the rates of denitrification and denitrification in the sediments of an eutrophic lake. Water Resources, vol. 18, no. 3, p 335-337.  
13. Morris JJ, Lenski RE, Zinser ER. 2012. The black queen hypothesis: Evolution of dependencies through adaptive gene loss. mBio 3(2). DOI: 10.1128/mBio.00036-12.
14. Yu Z, Yang J, Liu L. 2014. Denitrifier Community in the Oxygen Minimum Zone of a Subtropical Deep Reservoir. PLoS One 9(3): e92055.  
15. Palmer K, Horn MA. 2015. Denitrification Activity of a Remarkably Diverse Fen Denitrifier Community in Finnish Lapland Is N-Oxide Limited. PLoS One 10(4): e0123123.  

### Project_02_Self-assessment

To facilitate group assessment, please indicate your contributions to the following aspects of your final group reports. You should provide short statements under the headers relevant to your work. For example, you could specify that you "wrote the first draft", "wrote the code to accomplish X figure or result", "troubleshot code", "interpreted results", etc.

Include **separate** completed versions of this document for Project 1 **and** Project 2 in your portfolio repo; they do not need to be incorporated into the final knit portfolio html.

#### Results  
##### Anaysis   

##### Figures  

##### Writing  
I wrote the first draft for question 1, 2, 3, and 4.  

##### Editing  
I troubleshot the code for question 1, 2, 3, and 4 and figure formats.  

#### Discussion  
##### Literature research  
I looked at the literatures on the environmental and evolutionary reasons on the distributed metabolisms in the nitrogen cycle.

##### Writing  
I wrote the first draft for the proposal of environmental and evolutionary reasons on the distributed metabolisms in the nitrogen cycle. distribution. 

##### Analysis  
I wrote the code of the figure regarding oxygen in Saanich across depth.

##### Editing  
I edited the part of the environmental and evolutionary reasons on the distributed metabolisms.